<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">



<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation</title>





<link rel="stylesheet" href="/plosone/resource/compiled/asset_VQ5SPTCPS4VTM5BDJJ2NHA6OKDFQC6YN.css" />

  <!-- allows for  extra head tags -->



    <!--[if IE 8]>
  <link rel="stylesheet" type="text/css" href="/plosone/resource/css/ie.css"/>
    <![endif]-->

  <link media="print" rel="stylesheet" type="text/css"  href="/plosone/resource/css/print.css"/>
    <script type="text/javascript">
        var siteUrlPrefix = "/plosone/";
    </script>
<script src="/plosone/resource/compiled/asset_SC5JIUGEUPR4P4P6VBUINUVOVUSU3NRY.js"></script>
  <!--[if IE 8]>
  <script src="/plosone/resource/js/vendor/html5shiv.js"></script>
  <script src="/plosone/resource/js/vendor/respond.min.js"></script>
  <script src="/plosone/resource/js/vendor/rem.min.js"></script>
  <![endif]-->
  <link rel="shortcut icon" href="/plosone/resource/img/favicon.ico" type="image/x-icon"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>





  <meta name="description" content=" CLCA2 is a p53-, p63-inducible transmembrane protein that is frequently downregulated in breast cancer. It is induced during differentiation of human mammary epithelial cells, and its knockdown causes epithelial-to-mesenchymal transition (EMT). To determine how CLCA2 promotes epithelial differentiation, we searched for interactors using membrane dihybrid screening. We discovered a strong interaction with the cell junctional protein EVA1 (Epithelial V-like Antigen 1) and confirmed it by co-immunoprecipitation. Like CLCA2, EVA1 is a type I transmembrane protein that is regulated by p53 and p63. It is thought to mediate homophilic cell-cell adhesion in diverse epithelial tissues. We found that EVA1 is frequently downregulated in breast tumors and breast cancer cell lines, especially those of mesenchymal phenotype. Moreover, knockdown of EVA1 in immortalized human mammary epithelial cells (HMEC) caused EMT, implying that EVA1 is essential for epithelial differentiation. Both EVA1 and CLCA2 co-localized with E-cadherin at cell-cell junctions. The interacting domains were delimited by deletion analysis, revealing the site of interaction to be the transmembrane segment (TMS). The primary sequence of the CLCA2 TMS was found to be conserved in CLCA2 orthologs throughout mammals, suggesting that its interaction with EVA1 co-evolved with the mammary gland. A screen for other junctional interactors revealed that CLCA2 was involved in two different complexes, one with EVA1 and ZO-1, the other with beta catenin. Overexpression of CLCA2 caused downregulation of beta catenin and beta catenin-activated genes. Thus, CLCA2 links a junctional adhesion molecule to cytosolic signaling proteins that modulate proliferation and differentiation. These results may explain how attenuation of CLCA2 causes EMT and why CLCA2 and EVA1 are frequently downregulated in metastatic breast cancer cell lines. " />

  <meta name="keywords" content="Catenins,Breast cancer,Immunoprecipitation,Cell differentiation,Protein interactions,Breast tumors,Invasive ductal carcinoma,Epithelial cells" />

<meta name="citation_doi" content="10.1371/journal.pone.0147489" />
  <meta name="citation_author" content="Grace Ramena" />
        <meta name="citation_author_institution" content="Dept of Medical Microbiology, Immunology, and Cell Biology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America" />
  <meta name="citation_author" content="Yufang Yin" />
        <meta name="citation_author_institution" content="Dept of Pharmacology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America" />
  <meta name="citation_author" content="Yang Yu" />
        <meta name="citation_author_institution" content="Dept of Medical Microbiology, Immunology, and Cell Biology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America" />
  <meta name="citation_author" content="Vijay Walia" />
        <meta name="citation_author_institution" content="Laboratory of Cell and Developmental Signaling, National Cancer Institute-Frederick, Frederick, Maryland, 21702, United States of America" />
  <meta name="citation_author" content="Randolph C. Elble" />
        <meta name="citation_author_institution" content="Dept of Pharmacology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America" />
        <meta name="citation_author_institution" content="Simmons Cancer Institute, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America" />


  <meta name="citation_title" content="CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" />
  <meta itemprop="name" content="CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" />
  <meta name="citation_journal_title" content="PLOS ONE" />
  <meta name="citation_journal_abbrev" content="PLOS ONE" />
  <meta name="citation_date" content="Mar 1, 2016" />
<meta name="citation_firstpage" content="e0147489" />
<meta name="citation_issue" content="3" />
<meta name="citation_volume" content="11" />
<meta name="citation_issn" content="1932-6203" />

  <meta name="citation_publisher" content="Public Library of Science" />
  <meta name="citation_pdf_url" content="http://journals.plos.org/plosone/article/asset?id=10.1371/journal.pone.0147489.PDF">

<meta name="dc.identifier" content="10.1371/journal.pone.0147489" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" />
    <meta property="twitter:description" content=" CLCA2 is a p53-, p63-inducible transmembrane protein that is frequently downregulated in breast cancer. It is induced during differentiation of human mammary epithelial cells, and its knockdown causes epithelial-to-mesenchymal transition (EMT). To determine how CLCA2 promotes epithelial differentiation, we searched for interactors using membrane dihybrid screening. We discovered a strong interaction with the cell junctional protein EVA1 (Epithelial V-like Antigen 1) and confirmed it by co-immunoprecipitation. Like CLCA2, EVA1 is a type I transmembrane protein that is regulated by p53 and p63. It is thought to mediate homophilic cell-cell adhesion in diverse epithelial tissues. We found that EVA1 is frequently downregulated in breast tumors and breast cancer cell lines, especially those of mesenchymal phenotype. Moreover, knockdown of EVA1 in immortalized human mammary epithelial cells (HMEC) caused EMT, implying that EVA1 is essential for epithelial differentiation. Both EVA1 and CLCA2 co-localized with E-cadherin at cell-cell junctions. The interacting domains were delimited by deletion analysis, revealing the site of interaction to be the transmembrane segment (TMS). The primary sequence of the CLCA2 TMS was found to be conserved in CLCA2 orthologs throughout mammals, suggesting that its interaction with EVA1 co-evolved with the mammary gland. A screen for other junctional interactors revealed that CLCA2 was involved in two different complexes, one with EVA1 and ZO-1, the other with beta catenin. Overexpression of CLCA2 caused downregulation of beta catenin and beta catenin-activated genes. Thus, CLCA2 links a junctional adhesion molecule to cytosolic signaling proteins that modulate proliferation and differentiation. These results may explain how attenuation of CLCA2 causes EMT and why CLCA2 and EVA1 are frequently downregulated in metastatic breast cancer cell lines. " />
    <meta property="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0147489.g013" />

<meta property="og:type" content="article" />
  <meta property="og:url" content="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147489" />
  <meta property="og:title" content="CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" />
  <meta property="og:description" content=" CLCA2 is a p53-, p63-inducible transmembrane protein that is frequently downregulated in breast cancer. It is induced during differentiation of human mammary epithelial cells, and its knockdown causes epithelial-to-mesenchymal transition (EMT). To determine how CLCA2 promotes epithelial differentiation, we searched for interactors using membrane dihybrid screening. We discovered a strong interaction with the cell junctional protein EVA1 (Epithelial V-like Antigen 1) and confirmed it by co-immunoprecipitation. Like CLCA2, EVA1 is a type I transmembrane protein that is regulated by p53 and p63. It is thought to mediate homophilic cell-cell adhesion in diverse epithelial tissues. We found that EVA1 is frequently downregulated in breast tumors and breast cancer cell lines, especially those of mesenchymal phenotype. Moreover, knockdown of EVA1 in immortalized human mammary epithelial cells (HMEC) caused EMT, implying that EVA1 is essential for epithelial differentiation. Both EVA1 and CLCA2 co-localized with E-cadherin at cell-cell junctions. The interacting domains were delimited by deletion analysis, revealing the site of interaction to be the transmembrane segment (TMS). The primary sequence of the CLCA2 TMS was found to be conserved in CLCA2 orthologs throughout mammals, suggesting that its interaction with EVA1 co-evolved with the mammary gland. A screen for other junctional interactors revealed that CLCA2 was involved in two different complexes, one with EVA1 and ZO-1, the other with beta catenin. Overexpression of CLCA2 caused downregulation of beta catenin and beta catenin-activated genes. Thus, CLCA2 links a junctional adhesion molecule to cytosolic signaling proteins that modulate proliferation and differentiation. These results may explain how attenuation of CLCA2 causes EMT and why CLCA2 and EVA1 are frequently downregulated in metastatic breast cancer cell lines. " />
  <meta property="og:image" content="http://dx.plos.org/10.1371/journal.pone.0147489.g013" />

    <meta name="citation_reference" content="
      citation_title=Residual breast cancers after conventional therapy display mesenchymal as well as tumor-initiating features;
      citation_author=CJ Creighton;citation_author=X Li;citation_author=M Landis;citation_author=JM Dixon;citation_author=VM Neumeister;citation_author=A Sjolund;
      citation_journal_title=Proc Natl Acad Sci USA;
      citation_volume=106;
      citation_number=106;
      citation_pages=13820-13825;
      citation_date=2009;" />
    <meta name="citation_reference" content="
      citation_title=Loss of E-cadherin promotes metastasis via multiple downstream transcriptional pathways;
      citation_author=TT Onder;citation_author=PB Gupta;citation_author=SA Mani;citation_author=J Yang;citation_author=ES Lander;citation_author=RA Weinberg;
      citation_journal_title=Cancer Res;
      citation_volume=68;
      citation_number=68;
      citation_pages=3645-3654;
      citation_date=2008;" />
    <meta name="citation_reference" content="
      citation_title=The epithelial-mesenchymal transition generates cells with properties of stem cells;
      citation_author=S Mani;citation_author=W Guo;citation_author=M Liao;citation_author=E Eaton;citation_author=A Ayyanan;citation_author=A Zhou;
      citation_journal_title=Cell;
      citation_volume=133;
      citation_number=133;
      citation_pages=704-715;
      citation_date=2008;" />
    <meta name="citation_reference" content="
      citation_title=TGFbeta/TNF(alpha)-mediated epithelial-mesenchymal transition generates breast cancer stem cells with a claudin-low phenotype;
      citation_author=MK Asiedu;citation_author=JN Ingle;citation_author=MD Behrens;citation_author=DC Radisky;citation_author=KL Knutson;
      citation_journal_title=Cancer Res;
      citation_volume=71;
      citation_number=71;
      citation_pages=4707-19;
      citation_date=2011;" />
    <meta name="citation_reference" content="
      citation_title=Convergence of Wnt, beta-catenin, and cadherin pathways;
      citation_author=WJ Nelson;citation_author=R Nusse;
      citation_journal_title=Science;
      citation_volume=303;
      citation_number=303;
      citation_pages=1483-1487;
      citation_date=2004;" />
    <meta name="citation_reference" content="
      citation_title=E-cadherin is inactivated in a majority of invasive human lobular breast cancers by truncation mutations throughout its extracellular domain;
      citation_author=G Berx;citation_author=AM Cleton-Jansen;citation_author=K Strumane;citation_author=WJ de Leeuw;citation_author=F Nollet;citation_author=F van Roy;
      citation_journal_title=Oncogene;
      citation_volume=13;
      citation_number=13;
      citation_pages=1919-1925;
      citation_date=1996;" />
    <meta name="citation_reference" content="
      citation_title=CLCAs—a family of metalloproteases of intriguing phylogenetic distribution and with cases of substituted catalytic sites;
      citation_author=A Lenart;citation_author=M Dudkiewicz;citation_author=M Grynberg;citation_author=K Pawłowski;
      citation_journal_title=PLoS One;
      citation_volume=8;
      citation_number=8;
      citation_pages=e62272;
      citation_date=2013;" />
    <meta name="citation_reference" content="
      citation_title=Discovery and cloning of the CLCA gene family;
      citation_author=A.D. Gruber;citation_author=R.C. Elble;citation_author=B. Pauli;
      citation_journal_title=Current Topics in Membranes;
      citation_volume=53;
      citation_number=53;
      citation_pages=367-387;
      citation_date=2002;" />
    <meta name="citation_reference" content="
      citation_title=Novel conserved hydrolase domain in the CLCA family of alleged calcium-activated chloride channels;
      citation_author=K. Pawłowski;citation_author=M. Lepistö;citation_author=N. Meinander;citation_author=U. Sivars;citation_author=M. Varga;citation_author=E. Wieslander;
      citation_journal_title=Proteins;
      citation_volume=63;
      citation_number=63;
      citation_pages=424-439;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=The putative chloride channel hCLCA2 has a single C-terminal transmembrane segment;
      citation_author=RC Elble;citation_author=V Walia;citation_author=HC Cheng;citation_author=CJ Connon;citation_author=L Mundhenk;citation_author=AD Gruber;
      citation_journal_title=J Biol Chem;
      citation_volume=281;
      citation_number=281;
      citation_pages=29448-54;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=hCLCA2 is a p53-Inducible Inhibitor of Breast Cancer Cell Proliferation;
      citation_author=V Walia;citation_author=M Ding;citation_author=S Kumar;citation_author=D Nie;citation_author=LS Premkumar;citation_author=RC Elble;
      citation_journal_title=Cancer Res;
      citation_volume=69;
      citation_number=69;
      citation_pages=6624-6633;
      citation_date=2009;" />
    <meta name="citation_reference" content="
      citation_title=CLCA2 tumour suppressor gene in 1p31 is epigenetically regulated in breast cancer;
      citation_author=X Li;citation_author=JK Cowell;citation_author=K Sossey-Alaoui;
      citation_journal_title=Oncogene;
      citation_volume=23;
      citation_number=23;
      citation_pages=1474-80;
      citation_date=2004;" />
    <meta name="citation_reference" content="
      citation_title=Tumorigenicity of human breast cancer is associated with loss of the Ca2+-activated chloride channel CLCA2;
      citation_author=AD Gruber;citation_author=BU Pauli;
      citation_journal_title=Cancer Res;
      citation_volume=59;
      citation_number=59;
      citation_pages=5488-5491;
      citation_date=1999;" />
    <meta name="citation_reference" content="
      citation_title=Genome-wide Profiling of AP-1–Regulated Transcription Provides Insights into the Invasiveness of Triple-Negative Breast Cancer;
      citation_author=Zhao Chunyan;citation_author=Q Yichun;citation_author=J Philip;citation_author=W Jian;citation_author=X Li;citation_author=R Pegah;
      citation_journal_title=Cancer Res;
      citation_volume=74;
      citation_number=74;
      citation_pages=3983-3994;
      citation_date=2014;" />
    <meta name="citation_reference" content="
      citation_title=CLCA2, a target of the p53 family, negatively regulates cancer cell migration and invasion;
      citation_author=Y Sasaki;citation_author=R Koyama;citation_author=R Maruyama;citation_author=T Hirano;citation_author=M Tamura;citation_author=J Sugisaka;
      citation_journal_title=Cancer Biol Ther;
      citation_volume=13;
      citation_number=13;
      citation_pages=1512-1521;
      citation_date=2012;" />
    <meta name="citation_reference" content="
      citation_title=Loss of breast epithelial marker hCLCA2 promotes epithelial to mesenchymal transition and metastasis;
      citation_author=V Walia;citation_author=Y Yang;citation_author=D Cao;citation_author=J Cheng;citation_author=K Rao;citation_author=B Hollier;
      citation_journal_title=Oncogene;
      citation_volume=31;
      citation_number=31;
      citation_pages=2237-2246;
      citation_date=2012;" />
    <meta name="citation_reference" content="
      citation_title=Epithelial-mesenchymal transition in breast cancer relates to the basal-like phenotype;
      citation_author=D Sarrio;citation_author=SM Rodriguez-Pinilla;citation_author=D Hardisson;citation_author=A Cano;citation_author=G Moreno-Bueno;citation_author=J Palacios;
      citation_journal_title=Cancer Res;
      citation_volume=68;
      citation_number=68;
      citation_pages=989-997;
      citation_date=2008;" />
    <meta name="citation_reference" content="
      citation_title=Enrichment for breast cancer cells with stem/progenitor properties by differential adhesion;
      citation_author=V. Walia;citation_author=RC Elble;
      citation_journal_title=Stem Cells &amp; Development;
      citation_volume=19;
      citation_number=19;
      citation_pages=1175-1182;
      citation_date=2010;" />
    <meta name="citation_reference" content="
      citation_title=Human ClCa1 modulates anionic conduction of calcium-dependent chloride currents;
      citation_author=M Hamann;citation_author=A Gibson;citation_author=N Davies;citation_author=A Jowett;citation_author=JP Walhin;citation_author=L Partington;
      citation_journal_title=J Physiol;
      citation_volume=587;
      citation_number=587;
      citation_pages=2255-2274;
      citation_date=2009;" />
    <meta name="citation_reference" content="
      citation_title=p53 induces distinct epigenetic states at its direct target promoters;
      citation_author=L Vrba;citation_author=DJ Junk;citation_author=P Novak;citation_author=BW Futscher;
      citation_journal_title=BMC Genomics;
      citation_volume=9;
      citation_number=9;
      citation_pages=486;
      citation_date=2008;" />
    <meta name="citation_reference" content="
      citation_title=New p63 targets in keratinocytes identified by a genome-wide approach;
      citation_author=MA Viganò;citation_author=J Lamartine;citation_author=B Testoni;citation_author=D Merico;citation_author=D Alotto;citation_author=C Castagnoli;
      citation_journal_title=EMBO J;
      citation_volume=25;
      citation_number=25;
      citation_pages=5105-16;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=p63 regulates an adhesion programme and cell survival in epithelial cells;
      citation_author=DK Carroll;citation_author=JS Carroll;citation_author=CO Leong;citation_author=F Cheng;citation_author=M Brown;citation_author=AA Mills;
      citation_journal_title=Nat Cell Biol;
      citation_volume=8;
      citation_number=8;
      citation_pages=551-61;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=Epithelial V-like antigen (EVA), a novel member of the immunoglobulin superfamily, expressed in embryonic epithelia with a potential role as homotypic adhesion molecule in thymus histogenesis;
      citation_author=M Guttinger;citation_author=F Sutti;citation_author=M Panigada;citation_author=S Porcellini;citation_author=B Merati;citation_author=M Mariani;
      citation_journal_title=J Cell Biol;
      citation_volume=141;
      citation_number=141;
      citation_pages=1061-71;
      citation_date=1998;" />
    <meta name="citation_reference" content="
      citation_title=Re-expression of detachment-inducible chloride channel mCLCA5 suppresses growth of metastatic breast cancer cells;
      citation_author=JR Beckley;citation_author=BU Pauli;citation_author=RC Elble;
      citation_journal_title=J.Biol.Chem;
      citation_volume=279;
      citation_number=279;
      citation_pages=41634-41641;
      citation_date=2004;" />
    <meta name="citation_reference" content="
      citation_title=Loss of CLCA4 Promotes Epithelial-to-Mesenchymal Transition in Breast Cancer Cells;
      citation_author=Y Yu;citation_author=V Walia;citation_author=RC Elble;
      citation_journal_title=PLoS ONE;
      citation_volume=8;
      citation_number=8;
      citation_pages=e83943;
      citation_date=2013;" />
    <meta name="citation_reference" content="
      citation_title=Epithelial V-Like Antigen Mediates Efficacy of AntiAlpha4 Integrin Treatment in a Mouse Model of Multiple Sclerosis;
      citation_author=E Wright;citation_author=R Kusha;citation_author=H Nicholas;citation_author=L Stefan;citation_author=M.D Carrithers;
      citation_journal_title=PLoS ONE;
      citation_volume=8;
      citation_number=8;
      citation_pages=e70954;
      citation_date=2013;" />
    <meta name="citation_reference" content="
      citation_title=Endothelial cell diversity revealed by global expression profiling;
      citation_author=Chi Jen-Tsan;citation_author=YC Howard;citation_author=H Guttorm;citation_author=LJ Frode;citation_author=GT Olga;citation_author=SC Dustin;
      citation_journal_title=Proc Natl Acad Sci USA;
      citation_volume=100;
      citation_number=100;
      citation_pages=10623-10628;
      citation_date=2003;" />
    <meta name="citation_reference" content="
      citation_title=In vitro propagation and transcriptional profiling of human mammary stem/progenitor cells;
      citation_author=G Dontu;citation_author=MA Wissam;citation_author=MF Jessica;citation_author=WJ Kyle;citation_author=Michael F. Clarke;citation_author=Mari J. Kawamura;
      citation_journal_title=Genes Dev;
      citation_volume=17;
      citation_number=17;
      citation_pages=1253-1270;
      citation_date=2003;" />
    <meta name="citation_reference" content="
      citation_title=In vitro multipotent differentiation and barrier function of a human mammary epithelium;
      citation_author=AM Marshall;citation_author=PP Vaibhav;citation_author=AS Maureen;citation_author=DH Nelson;
      citation_journal_title=Cell and Tissue Res;
      citation_volume=335;
      citation_number=335;
      citation_pages=383-395;
      citation_date=2009;" />
    <meta name="citation_reference" content="
      citation_title=Core epithelial-to-mesenchymal transition interactome gene-expression signature is associated with claudin-low and metaplastic breast cancer subtypes;
      citation_author=JH Taube;citation_author=JI Herschkowitz;citation_author=K Komurov;citation_author=AY Zhou;citation_author=S Gupta;citation_author=J Yang;
      citation_journal_title=Proc Natl Acad Sci USA;
      citation_volume=107;
      citation_number=107;
      citation_pages=15449-15454;
      citation_date=2010;" />
    <meta name="citation_reference" content="
      citation_title=TRPS1 Targeting by miR-221/222 Promotes the Epithelial-to-Mesenchymal Transition in Breast Cancer;
      citation_author=S Stinson;citation_author=RL Mark;citation_author=TA Alex;citation_author=Y Nancy;citation_author=K Hyo-Jin;citation_author=OB Carol;
      citation_journal_title=Sci. Signal;
      citation_volume=4;
      citation_number=4;
      citation_pages=ra41;
      citation_date=2011;" />
    <meta name="citation_reference" content="
      citation_title=A dileucine motif targets E-cadherin to the basolateral cell surface in Madin-Darby canine kidney and LLC-PK1 epithelial cells;
      citation_author=KC Miranda;citation_author=T Khromykh;citation_author=P Christy;citation_author=TL Le;citation_author=CJ Gottardi;citation_author=AS Yap;
      citation_journal_title=J Biol Chem;
      citation_volume=276;
      citation_number=276;
      citation_pages=22565-72;
      citation_date=2001;" />
    <meta name="citation_reference" content="
      citation_title=Requirement of ZO-1 for the formation of belt-like adherens junctions during epithelial cell polarization;
      citation_author=J Ikenouchi;citation_author=K Umeda;citation_author=S Tsukita;citation_author=M Furuse;citation_author=S Tsukita;
      citation_journal_title=J Cell Biol;
      citation_volume=176;
      citation_number=176;
      citation_pages=779-86;
      citation_date=2007;" />
    <meta name="citation_reference" content="
      citation_title=Epithelial V-like antigen regulates permeability of the blood-CSF barrier;
      citation_author=G Chatterjee;citation_author=LM Carrithers;citation_author=MD Carrithers;
      citation_journal_title=Biochem Biophys Res Commun;
      citation_volume=372;
      citation_number=372;
      citation_pages=412-417;
      citation_date=2008;" />
    <meta name="citation_reference" content="
      citation_title=MAGUK proteins: structure and role in the tight junction;
      citation_author=L. Gonzalez-Mariscal;citation_author=A. Betanzos;citation_author=A. Avila-Flores;
      citation_journal_title=Semin. Cell Dev. Biol;
      citation_volume=11;
      citation_number=11;
      citation_pages=315-324;
      citation_date=2000;" />
    <meta name="citation_reference" content="
      citation_title=Cross-talk of WNT and FGF signaling pathways at GSK3beta to regulate beta-catenin and SNAIL signaling cascades;
      citation_author=M Katoh;citation_author=M Katoh;
      citation_journal_title=Cancer Biol Ther;
      citation_volume=5;
      citation_number=5;
      citation_pages=1059-1064;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=Wnt proteins prevent apoptosis of both uncommitted osteoblast progenitors and differentiated osteoblasts by beta-catenin-dependent and -independent signaling cascades involving src/ERK and phosphatidylinositol 3-kinase/AKT;
      citation_author=M Almeida;citation_author=L Han;citation_author=T Bellido;citation_author=SC Manolagas;citation_author=S Kousteni;
      citation_journal_title=J Biol Chem;
      citation_volume=280;
      citation_number=280;
      citation_pages=41342-41351;
      citation_date=2005;" />
    <meta name="citation_reference" content="
      citation_title=Transcriptional activation of cyclooxygenase-2 in Wnt-1- transformed mouse mammary epithelial cells;
      citation_author=LR Howel;citation_author=K Subbaramaiah;citation_author=WJ Chung;citation_author=AJ Dannenberg;citation_author=AM Brown;
      citation_journal_title=Cancer Res;
      citation_volume=59;
      citation_number=59;
      citation_pages=1572-577;
      citation_date=1999;" />
    <meta name="citation_reference" content="
      citation_title=Beta-catenin regulates expression of cyclin D1 in colon carcinoma cells;
      citation_author=O Tetsu;citation_author=F McCormick;
      citation_journal_title=Nature;
      citation_volume=398;
      citation_number=398;
      citation_pages=422-426;
      citation_date=1999;" />
    <meta name="citation_reference" content="
      citation_title=ITF-2, a downstream target of the Wnt/TCF pathway, is activated in human cancers with beta-catenin defects and promotes neoplastic transformation;
      citation_author=FT Kolligs;citation_author=MT Nieman;citation_author=I Winer;citation_author=G Hu;citation_author=D Van Mater;citation_author=Y Feng;
      citation_journal_title=Cancer Cell;
      citation_volume=1;
      citation_number=1;
      citation_pages=145-155;
      citation_date=2002;" />
    <meta name="citation_reference" content="
      citation_title=Targeting Wnt signaling in colorectalcancer. A Review in the Theme: Cell Signaling: Proteins, Pathways and Mechanisms;
      citation_author=L Novellasdemunt;citation_author=P Antas;citation_author=VS Li;
      citation_journal_title=Am J Physiol Cell Physiol;
      citation_volume=309;
      citation_number=309;
      citation_pages=C511-21;
      citation_date=2015;" />
    <meta name="citation_reference" content="
      citation_title=Roles and modes of action of nectins in cell-cell adhesion;
      citation_author=K Irie;citation_author=K Shimizu;citation_author=T Sakisaka;citation_author=W Ikeda;citation_author=Y Takai;
      citation_journal_title=Semin. Cell Dev. Biol;
      citation_volume=15;
      citation_number=15;
      citation_pages=643-656;
      citation_date=2004;" />
    <meta name="citation_reference" content="
      citation_title=The role of the cell adhesion molecule vomorulin in the formation and maintenance of the epithelial junctional complex;
      citation_author=B Gumbiner;citation_author=B. Stevenson;citation_author=A. Grimaldi;
      citation_journal_title=J. Cell Biol;
      citation_volume=107;
      citation_number=107;
      citation_pages=1575-1587;
      citation_date=1988;" />
    <meta name="citation_reference" content="
      citation_title=Cadherins in development: cell adhesion, sorting, and tissue morphogenesis;
      citation_author=J.M. Halbleib;citation_author=W.J. Nelson;
      citation_journal_title=Genes Dev;
      citation_volume=20;
      citation_number=20;
      citation_pages=3199-3214;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=Cell-to-cell adherens junction formation and actin filament organization: similarities and differences between non-polarized fibroblasts and polarized epithelial cells;
      citation_author=S Yonemura;citation_author=A M Itoh;citation_author=Nagafuchi;citation_author=S Tsukita;
      citation_journal_title=J. Cell Sci;
      citation_volume=108;
      citation_number=108;
      citation_pages=127-14;
      citation_date=1995;" />
    <meta name="citation_reference" content="
      citation_title=Hyperproliferation and defects in epithelial polarity on conditional ablation of a-catenin in skin;
      citation_author=V Vasioukhin;citation_author=C Bauer;citation_author=L Degenstein;citation_author=B Wise;citation_author=E Fuchs;
      citation_journal_title=Cell;
      citation_volume=104;
      citation_number=104;
      citation_pages=605-17;
      citation_date=2001;" />
    <meta name="citation_reference" content="
      citation_title=Alpha-catenin is a molecular switch that binds E-cadherin-beta-catenin and regulates actin-filament assembly;
      citation_author=F Drees;citation_author=S Pokutta;citation_author=S Yamada;citation_author=WJ Nelson;citation_author=WI Weis;
      citation_journal_title=Cell;
      citation_volume=123;
      citation_number=123;
      citation_pages=903-915;
      citation_date=2005;" />
    <meta name="citation_reference" content="
      citation_title=Calcium modulation of adherens and tight junction function: a potential mechanism for blood-brain barrier disruption after stroke;
      citation_author=RC Brown;citation_author=TP Davis;
      citation_journal_title=Stroke;
      citation_volume=33;
      citation_number=33;
      citation_pages=1706-11;
      citation_date=2002;" />
    <meta name="citation_reference" content="
      citation_title=Dependence of epithelial intercellular junction biogenesis on thapsigargin-sensitive intracellular calcium stores;
      citation_author=RO Stuart;citation_author=A Sun;citation_author=KT Bush;citation_author=SK Nigam;
      citation_journal_title=J Biol Chem;
      citation_volume=271;
      citation_number=271;
      citation_pages=13636-41;
      citation_date=1996;" />
    <meta name="citation_reference" content="
      citation_title=Secreted CLCA1 modulates TMEM16A to activate Ca(2+)-dependent chloride currents in human cells;
      citation_author=M Sala-Rabanal;citation_author=Z Yurtsever;citation_author=CG Nichols;citation_author=TJ Brett;
      citation_journal_title=Elife;
      citation_volume=4;
      citation_number=4;
      citation_date=2015;" />
    <meta name="citation_reference" content="
      citation_title=Self-cleavage of human CLCA1 protein by a novel internal metalloprotease domain controls calcium-activated chloride channel activation;
      citation_author=Z Yurtsever;citation_author=M Sala-Rabanal;citation_author=DT Randolph;citation_author=SM Scheaffer;citation_author=WT Roswit;citation_author=YG Alevy;citation_author=AC Patel;citation_author=RF Heier;citation_author=AG Romero;citation_author=CG Nichols;citation_author=MJ Holtzman;citation_author=TJ Brett;
      citation_journal_title=J Biol Chem;
      citation_volume=287;
      citation_number=287;
      citation_pages=42138-49;
      citation_date=2012;" />
    <meta name="citation_reference" content="
      citation_title=Secreted hCLCA1 is a signaling molecule that activates airway macrophages;
      citation_author=JC Ching;citation_author=L Lobanova;citation_author=ME Loewen;
      citation_journal_title=PLoS One;
      citation_volume=8;
      citation_number=8;
      citation_pages=e83130;
      citation_date=2013;" />
    <meta name="citation_reference" content="
      citation_title=CLCA2 as a novel immunohistochemical marker for differential diagnosis of squamous cell carcinoma from adenocarcinoma of the lung;
      citation_author=K Shinmura;citation_author=H Igarashi;citation_author=H Kato;citation_author=Y Kawanishi;citation_author=Y Inoue;citation_author=S Nakamura;
      citation_journal_title=Dis Markers;
      citation_date=2014;" />
    <meta name="citation_reference" content="
      citation_title=Gene expression and immunolocalisation of a calcium-activated chloride channel during the stratification of cultivated and developing corneal epithelium;
      citation_author=CJ Connon;citation_author=S Kawasaki;citation_author=M Liles;citation_author=N Koizumi;citation_author=K Yamasaki;citation_author=T Nakamura;citation_author=AJ Quantock;citation_author=S Kinoshita;
      citation_journal_title=Cell Tissue Res;
      citation_volume=323;
      citation_number=323;
      citation_pages=177-182;
      citation_date=2006;" />
    <meta name="citation_reference" content="
      citation_title=E-cadherin-integrin crosstalk in cancer invasion and metastasis;
      citation_author=M Canel;citation_author=A Serrels;citation_author=MC Frame;citation_author=VG Brunton;
      citation_journal_title=J Cell Sci;
      citation_volume=126;
      citation_number=126;
      citation_pages=393-401;
      citation_date=2013;" />
    <meta name="citation_reference" content="
      citation_title=ANO1 interacts with EGFR and correlates with sensitivity to EGFR-targeting therapy in head and neck cancer;
      citation_author=A Bill;citation_author=A Gutierrez;citation_author=S Kulkarni;citation_author=C Kemp;citation_author=D Bonenfant;citation_author=H Voshol;
      citation_journal_title=Oncotarget;
      citation_volume=6;
      citation_number=6;
      citation_pages=9173-88;
      citation_date=2015;" />
    <meta name="citation_reference" content="
      citation_title=Junctional adhesion molecule, a novel member of the immunoglobulin superfamily that distributes at intercellular junctions and modulates monocyte transmigration;
      citation_author=I Martìn-Padura;citation_author=S Lostaglio;citation_author=M Schneemann;citation_author=L Williams;citation_author=M Romano;citation_author=P Fruscella;
      citation_journal_title=J Cell Biol;
      citation_volume=142;
      citation_number=142;
      citation_pages=117-127;
      citation_date=1998;" />
    <meta name="citation_reference" content="
      citation_title=Gene expression profiling of breast cell lines identifies potential new basal markers;
      citation_author=E Charafe-Jauffret;citation_author=C Ginestier;citation_author=F Monville;citation_author=P Finetti;citation_author=J Adélaïde;citation_author=N Cervera;citation_author=S Fekairi;citation_author=L Xerri;citation_author=J Jacquemier;citation_author=D Birnbaum;citation_author=F Bertucci;
      citation_journal_title=Oncogene;
      citation_volume=25;
      citation_number=25;
      citation_pages=2273-84;
      citation_date=2006;" />


<!-- DoubleClick overall ad setup script -->
<script type='text/javascript'>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
  (function() {
    var gads = document.createElement('script');
    gads.async = true;
    gads.type = 'text/javascript';
    var useSSL = 'https:' == document.location.protocol;
    gads.src = (useSSL ? 'https:' : 'http:') +
        '//www.googletagservices.com/tag/js/gpt.js';
    var node = document.getElementsByTagName('script')[0];
    node.parentNode.insertBefore(gads, node);
  })();
</script>

<!-- DoubleClick ad slot setup script -->
  <script id="doubleClickSetupScript" type='text/javascript'>
    googletag.cmd.push(function() {
  googletag.defineSlot('/75507958/PONE_728x90_ATF', [728, 90], 'div-gpt-ad-1458247671871-0').addService(googletag.pubads());
  googletag.defineSlot('/75507958/PONE_160x600_BTF', [160, 600], 'div-gpt-ad-1458247671871-1').addService(googletag.pubads());
      googletag.pubads().enableSingleRequest();
      googletag.enableServices();
    });
  </script>
<script type="text/javascript">
    var WombatConfig = WombatConfig || {};
    WombatConfig.resourcePath = "/plosone/resource/";
    WombatConfig.imgPath = "/plosone/resource/img/";
    WombatConfig.journalKey = "PLoSONE";
    WombatConfig.figurePath = "/plosone/article/figure/image";
    WombatConfig.metrics = WombatConfig.metrics || {};
    WombatConfig.metrics.referenceUrl      = "http://lagotto.io/plos";
    WombatConfig.metrics.googleScholarUrl  = "http://scholar.google.com/scholar";
    WombatConfig.metrics.googleScholarCitationUrl  = WombatConfig.metrics.googleScholarUrl + "?hl=en&lr=&cites=";
    WombatConfig.metrics.crossrefUrl  = "http://www.crossref.org";
</script><script src="//code.jquery.com/jquery-2.1.4.min.js" ></script>
<script>window.jQuery || document.write('<script src="/plosone/resource/js/vendor/jquery-2.1.4.min.js""><\/script>')</script>







<!--For Google Tag manager to be able to track site information  -->
<script>

  dataLayer = [{
    'mobileSite': 'false',
    'desktopSite': 'true'
  }];

</script>

</head>




<body class="article plosone">

<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-TP26BH"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-TP26BH');</script>

<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-MQQMGF"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-MQQMGF');</script>
<!-- End Google Tag Manager -->
<header>
  <div id="topslot" class="head-top">
  <div class="center">
<div class="title">Advertisement</div>
<!-- DoubleClick Ad Zone -->
  <div class='advertisement' id='div-gpt-ad-1458247671871-0' style='width:728px; height:90px;'>
    <script type='text/javascript'>
      googletag.cmd.push(function() { googletag.display('div-gpt-ad-1458247671871-0'); });
    </script>
  </div>
</div>
  </div>

  <div id="user" class="nav">
    <ul class="nav-user">

 


      <li ><a href="http://www.plos.org">plos.org</a></li>


        <li ><a href="https://community.plos.org/registration/new">create account</a></li>


          <li class="highlighted"><a href="/plosone/user/secure/login?page=%2Fplosone%2Farticle%3Fid%3D10.1371%2Fjournal.pone.0147489">sign in</a></li>

    </ul>
  </div>
  <div id="pagehdr">

    <nav class="nav-main">




<h1 class="logo">
  <a href="/plosone/.">PLOS ONE</a>
</h1>

<section class="top-bar-section"> 

<ul class="nav-elements">


    <li class="multi-col-parent menu-section-header has-dropdown" id="publish">
    Publish
      <div class="dropdown mega ">
        <ul class="multi-col" id="publish-dropdown-list">

  <li class="menu-section-header " id="submissions">
    <span class="menu-section-header-title">  Submissions </span>

    <ul class="menu-section "
        id="submissions-dropdown-list">
      <li>
    <a href="/plosone/s/getting-started" >Getting Started</a>
  </li>

      <li>
    <a href="/plosone/s/submission-guidelines" >Submission Guidelines</a>
  </li>

      <li>
    <a href="/plosone/s/figures" >Figures</a>
  </li>

      <li>
    <a href="/plosone/s/tables" >Tables</a>
  </li>

      <li>
    <a href="/plosone/s/supporting-information" >Supporting Information</a>
  </li>

      <li>
    <a href="/plosone/s/latex" >LaTeX</a>
  </li>

      <li>
    <a href="/plosone/s/revising-your-manuscript" >Revising Your Manuscript</a>
  </li>

      <li>
    <a href="/plosone/s/submit-now" >Submit Now</a>
  </li>

    </ul>

  </li>


  <li class="menu-section-header " id="policies">
    <span class="menu-section-header-title">  Policies </span>

    <ul class="menu-section "
        id="policies-dropdown-list">
      <li>
    <a href="/plosone/s/best-practices-in-research-reporting" >Best Practices in Research Reporting</a>
  </li>

      <li>
    <a href="/plosone/s/human-subjects-research" >Human Subjects Research</a>
  </li>

      <li>
    <a href="/plosone/s/animal-research" >Animal Research</a>
  </li>

      <li>
    <a href="/plosone/s/competing-interests" >Competing Interests</a>
  </li>

      <li>
    <a href="/plosone/s/disclosure-of-funding-sources" >Disclosure of Funding Sources</a>
  </li>

      <li>
    <a href="/plosone/s/content-license" >Content License</a>
  </li>

      <li>
    <a href="/plosone/s/data-availability" >Data Availability</a>
  </li>

      <li>
    <a href="/plosone/s/materials-and-software-sharing" >Materials and Software Sharing</a>
  </li>

      <li>
    <a href="/plosone/s/ethical-publishing-practice" >Ethical Publishing Practice</a>
  </li>

      <li>
    <a href="/plosone/s/authorship" >Authorship</a>
  </li>

      <li>
    <a href="/plosone/s/downloads-and-translations" >Downloads and Translations</a>
  </li>

    </ul>

  </li>


  <li class="menu-section-header " id="manuscript-review-and-publication">
    <span class="menu-section-header-title">  Manuscript Review and Publication </span>

    <ul class="menu-section "
        id="manuscript-review-and-publication-dropdown-list">
      <li>
    <a href="/plosone/s/criteria-for-publication" >Criteria for Publication</a>
  </li>

      <li>
    <a href="/plosone/s/editorial-and-peer-review-process" >Editorial and Peer Review Process</a>
  </li>

      <li>
    <a href="/plosone/s/reviewer-guidelines" >Reviewer Guidelines</a>
  </li>

      <li>
    <a href="/plosone/s/accepted-manuscripts" >Accepted Manuscripts</a>
  </li>

      <li>
    <a href="/plosone/s/corrections-and-retractions" >Corrections and Retractions</a>
  </li>

      <li>
    <a href="/plosone/s/comments" >Comments</a>
  </li>

      <li>
    <a href="/plosone/s/article-level-metrics" >Article-Level Metrics</a>
  </li>

    </ul>

  </li>
        </ul>
          <div class="calloutcontainer">


  <h3 class="callout-headline">Submit Your Manuscript</h3>

  <div class="action-contain">
  <p class="callout-content">
  Discover a faster, simpler path to publishing in a high-quality journal. <em>PLOS ONE</em> promises fair, rigorous peer review,
  broad scope, and wide readership – a perfect fit for your research every time.
  </p>
  <p class="button-contain special">
    <a class="button button-default" href="http://www.plosone.org/static/publish">
     Learn More
    </a>
    <a class="button-link" href="http://www.editorialmanager.com/pone/default.asp">
      Submit Now
    </a>
  </p>
  </div>  <!-- opens in siteMenuCalloutDescription -->


          </div>
      </div>
    </li>



  <li class="menu-section-header has-dropdown " id="about">
    <span class="menu-section-header-title">  About </span>

    <ul class="menu-section dropdown "
        id="about-dropdown-list">
      <li>
    <a href="/plosone/static/publish" >Why Publish with PLOS ONE</a>
  </li>

      <li>
    <a href="/plosone/s/journal-information" >Journal Information</a>
  </li>

      <li>
    <a href="/plosone/s/staff-editors" >Staff Editors</a>
  </li>

      <li>
    <a href="/plosone/static/editorial-board" >Editorial Board</a>
  </li>

      <li>
    <a href="/plosone/s/section-editors" >Section Editors</a>
  </li>

      <li>
    <a href="/plosone/s/advisory-groups" >Advisory Groups</a>
  </li>

      <li>
    <a href="/plosone/s/publishing-information" >Publishing Information</a>
  </li>

      <li>
    <a href="/plosone/s/publication-fees" >Publication Fees</a>
  </li>

      <li>
    <a href="/plosone/s/press-and-media" >Press and Media</a>
  </li>

      <li>
    <a href="/plosone/s/contact" >Contact</a>
  </li>

    </ul>

  </li>

  <li data-js-tooltip-hover="trigger" class="subject-area menu-section-header">
  Browse
  </li>




<li id="navsearch" class="head-search">
  <form name="searchForm" action="/plosone/search" method="get">
        <fieldset>
            <legend>Search</legend>
            <label for="search">Search</label>
            <input id="search" type="text" name="q" placeholder="Search" required/>
            <button id="headerSearchButton" type="submit"><span class="search-icon"></span></button>

        </fieldset>
        <input type="hidden" name="filterJournals" value="PLoSONE"/>
    </form>

    <a id="advSearch"
       href="/plosone/search">
      advanced search
    </a>

  </li>




      </ul>     
      </section>  
    </nav>
  </div>

</header><section id="taxonomyContainer">
<div id="taxonomy-browser" class="areas" data-search-url="/plosone/browse">
  <div class="wrapper">
    <div class="taxonomy-header">
      Browse Subject Areas
      <div id="subjInfo">?</div>
      <div id="subjInfoText">
        <p>Click through the PLOS taxonomy to find articles in your field.</p>
        <p>For more information about PLOS Subject Areas, click
          <a href="/plosone/s/help-using-this-site#loc-subject-areas">here</a>.
        </p>
      </div>
    </div>
    <div class="levels">
      <div class="levels-container cf">
        <div class="levels-position"></div>
      </div>
      <a href="#" class="prev"></a>
      <a href="#" class="next active"></a>
    </div>
  </div>
  <div class="taxonomy-browser-border-bottom"></div>
</div></section>
<main> <div class="set-grid">
<header class="title-block">

<ul id="almSignposts" class="signposts">
  <li id="loadingMetrics">
    <p>Loading metrics</p>
  </li>
</ul>

<script type="text/template" id="signpostsGeneralErrorTemplate">
  <li id="metricsError">Article metrics are unavailable at this time. Please try again later.</li>
</script>

<script type="text/template" id="signpostsNewArticleErrorTemplate">
  <li></li><li></li><li id="tooSoon">Article metrics are unavailable for recently published articles.</li>
</script>

<script type="text/template" id="signpostsTemplate">
    <li id="almSaves">
      <%= s.numberFormat(saveCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#savedHeader">Save</a>
        <p class="saves-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#savedHeader">Total Mendeley and CiteULike bookmarks.</a></p>
      </div>
    </li>

    <li id="almCitations">
      <%= s.numberFormat(citationCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#citedHeader">Citation</a>
        <p class="citations-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#citedHeader">Paper's citation count computed by Scopus.</a></p>
      </div>
    </li>

    <li id="almViews">
      <%= s.numberFormat(viewCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#viewedHeader">View</a>
        <p class="views-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#viewedHeader">Sum of PLOS and PubMed Central page views and downloads.</a></p>
      </div>
    </li>

    <li id="almShares">
      <%= s.numberFormat(shareCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#discussedHeader">Share</a>
        <p class="shares-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489#discussedHeader">Sum of Facebook and Twitter activity.</a></p>
      </div>
    </li>
</script>
    <div class="article-meta">

<div class="classifications">
  <p class="license-short" id="licenseShort">Open Access</p>
  <p class="peer-reviewed" id="peerReviewed">Peer-reviewed</p>

<div class="article-type" >
  <p class="type-article" id="artType">Research Article</p>
</div>


</div>


    </div>
    <div class="article-title-etc">


<div class="title-authors">
  <h1 id="artTitle">CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation</h1>
<ul class="author-list clearfix"  data-js-tooltip="tooltip_container" id="author-list">



<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="0" class="author-name" >
Grace Ramena,</a>    <div id="author-meta-0" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-0"><span class="type">Affiliation</span>
    Dept of Medical Microbiology, Immunology, and Cell Biology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose0"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="1" class="author-name" >
Yufang Yin,</a>    <div id="author-meta-1" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-1"><span class="type">Affiliation</span>
    Dept of Pharmacology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose1"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="2" class="author-name" >
Yang Yu,</a>    <div id="author-meta-2" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-2"><span class="type">Affiliation</span>
    Dept of Medical Microbiology, Immunology, and Cell Biology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose2"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="3" class="author-name" >
Vijay Walia,</a>    <div id="author-meta-3" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-3"><span class="type">Affiliation</span>
    Laboratory of Cell and Developmental Signaling, National Cancer Institute-Frederick, Frederick, Maryland, 21702, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose3"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="4" class="author-name" >
Randolph C. Elble <span class="email">  </span></a>    <div id="author-meta-4" class="author-info" data-js-tooltip="tooltip_target">

  
  <p id="authCorresponding-4"> <span class="email">* E-mail:</span> <a href="mailto:relble2@siumed.edu">relble2@siumed.edu</a></p>
  <p id="authAffiliations-4"><span class="type">Affiliations</span>
    Dept of Pharmacology, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America, 
    Simmons Cancer Institute, Southern Illinois University School of Medicine, Springfield, Illinois, 62794, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose4"> &#x02A2F; </a>
    </div>
</li>

</ul>

</div>


<div id="floatTitleTop" data-js-floater="title_author" class="float-title">
  <div class="set-grid">
    <div class="float-title-inner">
      <h1>CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation</h1>

<ul id="floatAuthorList" data-js-floater="floated_authors">

  <li data-float-index="1">Grace Ramena,&nbsp;

  </li>
  <li data-float-index="2">Yufang Yin,&nbsp;

  </li>
  <li data-float-index="3">Yang Yu,&nbsp;

  </li>
  <li data-float-index="4">Vijay Walia,&nbsp;

  </li>
  <li data-float-index="5">Randolph C. Elble

  </li>

</ul>



    </div>
    <div class="logo-close" id="titleTopCloser">
      <img src="/plosone/resource/img/logo.plos.95.png" alt="PLOS" />
      <div class="close-floater" title="close">x</div>
    </div>
  </div>
</div>

        <ul class="date-doi">
            <li id="artPubDate">Published: March 1, 2016</li>
            <li id="artDoi">
<a   href="http://dx.doi.org/10.1371/journal.pone.0147489">http://dx.doi.org/10.1371/journal.pone.0147489</a>
            </li>


        </ul>
    </div>
</header>  <section class="article-body">

<ul class="article-tabs">

    <li class="tab-title active" id="tabArticle">
      <a href="/plosone/article?id=10.1371%2Fjournal.pone.0147489" class="article-tab-1">Article</a>
  </li>


        <li class="tab-title " id="tabAuthors">
      <a href="/plosone/article/authors?id=10.1371%2Fjournal.pone.0147489" class="article-tab-2">Authors</a>
  </li>


    <li class="tab-title " id="tabMetrics">
      <a href="/plosone/article/metrics?id=10.1371%2Fjournal.pone.0147489" class="article-tab-3">Metrics</a>
  </li>

    <li class="tab-title " id="tabComments">
      <a href="/plosone/article/comments?id=10.1371%2Fjournal.pone.0147489" class="article-tab-4">Comments</a>
  </li>

    <li class="tab-title " id="tabRelated">
      <a href="/plosone/article/related?id=10.1371%2Fjournal.pone.0147489" class="article-tab-5">Related Content</a>
  </li>

</ul>

    <div class="article-container">

<div id="nav-article">
  <ul class="nav-secondary">

    <li class="nav-comments" id="nav-comments">
      <a href="article/comments?id=10.1371/journal.pone.0147489">Reader Comments (0)</a>
    </li>

    <li class="nav-media" id="nav-media" data-doi="10.1371/journal.pone.0147489">
      <a href="/plosone/article/related?id=10.1371%2Fjournal.pone.0147489">
        Media Coverage <span id="media-coverage-count"></span>
      </a>
    </li>

    <li id="nav-figures"><a href="#" data-doi="10.1371/journal.pone.0147489">Figures</a></li>
  </ul>
</div>
<div id="figure-lightbox-container"></div>

<script id="figure-lightbox-template" type="text/template">
  <div id="figure-lightbox" class="reveal-modal full" data-reveal aria-hidden="true"
       role="dialog">
    <div class="lb-header">
      <h1 id="lb-title"><%= articleTitle %></h1>

      <div id="lb-authors">
        <% authorList.split(',').forEach(function (author) { %>
        <span><%= author.trim() %></span>
        <% }) %>
      </div>

      <div class="lb-close" title="close">&nbsp;</div>
    </div>
    <div class="img-container">
      <div class="loader"> <i class="fa-spinner"></i> </div>
      <img class="main-lightbox-image" src=""/>
      <aside id="figures-list">
        <% figureList.each(function (ix, figure) { %>
        <div class="change-img" data-doi="<%= figure.getAttribute('data-doi') %>">
          <img class="aside-figure" src="/plosone/article/figure/image?size=inline&id=<%= figure.getAttribute('data-doi') %>" />
        </div>
        <% }) %>
        <div class="dummy-figure">
        </div>
      </aside>
    </div>
    <div id="lightbox-footer">

      <div id="btns-container" class="lightbox-row <% if(figureList.length <= 1) { print('one-figure-only') } %>">
        <div class="fig-btns-container reset-zoom-wrapper left">
          <span class="fig-btn reset-zoom-btn">Reset zoom</span>
        </div>
        <div class="zoom-slider-container">
          <div class="range-slider-container">
            <span id="lb-zoom-min"></span>
            <div class="range-slider round" data-slider data-options="start: 20; end: 200; initial: 20;">
              <span class="range-slider-handle" role="slider" tabindex="0"></span>
              <span class="range-slider-active-segment"></span>
              <input type="hidden">
            </div>
            <span id="lb-zoom-max"></span>
          </div>
        </div>
        <% if(figureList.length > 1) { %>
        <div class="fig-btns-container">
          <span class="fig-btn all-fig-btn"><i class="icon icon-all"></i> All Figures</span>
          <span class="fig-btn next-fig-btn"><i class="icon icon-next"></i> Next</span>
          <span class="fig-btn prev-fig-btn"><i class="icon icon-prev"></i> Previous</span>
        </div>
        <% } %>
      </div>
      <div id="image-context">
      </div>
    </div>
  </div>
</script>

<script id="image-context-template" type="text/template">
  <div class="footer-text">
    <div id="figure-description-wrapper">
      <div id="view-more-wrapper" style="<% descriptionExpanded? print('display:none;') : '' %>">
        <span id="figure-title"><%= title %></span>
        <p id="figure-description">
          <%= description %>&nbsp;&nbsp;
        </p>
        <span id="view-more">show more<i class="icon-arrow-right"></i></span>

      </div>
      <div id="view-less-wrapper" style="<% descriptionExpanded? print('display:inline-block;') : '' %>" >
        <span id="figure-title"><%= title %></span>
        <p id="full-figure-description">
          <%= description %>&nbsp;&nbsp;
          <span id="view-less">show less<i class="icon-arrow-left"></i></span>
        </p>
      </div>
    </div>
  </div>
  <div id="show-context-container">
    <a class="btn show-context" href="<%= showInContext(strippedDoi) %>">Show in Context</a>
  </div>
  <div id="download-buttons">
    <h3>Download:</h3>
    <div class="item">
      <a href="/plosone/article/figure/image?size=original&download=&id=<%= doi %>" title="original image">
        <span class="download-btn">TIFF</span>
      </a>
      <span class="file-size"><%= fileSizes.original %></span>
    </div>
    <div class="item">
      <a href="/plosone/article/figure/image?size=large&download=&id=<%= doi %>" title="large image">
        <span class="download-btn">PNG</span>
      </a>
      <span class="file-size"><%= fileSizes.large %></span>
    </div>
    <div class="item">
      <a href="/plosone/article/figure/powerpoint?id=<%= doi %>" title="PowerPoint slide">
        <span class="download-btn">PPT</span>
      </a>
    </div>

  </div>
</script>
      <div class="article-content">



<div id="figure-carousel-section">
  <h2>Figures</h2>

  <div id="figure-carousel">

    <div class="carousel-wrapper">
      <div class="slider">


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g001">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g001"
                     alt="Fig 1"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g002">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g002"
                     alt="Fig 2"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g003">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g003"
                     alt="Fig 3"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g004">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g004"
                     alt="Fig 4"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g005">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g005"
                     alt="Fig 5"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.t001">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.t001"
                     alt="Table 1"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g006">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g006"
                     alt="Fig 6"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g007">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g007"
                     alt="Fig 7"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g008">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g008"
                     alt="Fig 8"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g009">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g009"
                     alt="Fig 9"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g010">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g010"
                     alt="Fig 10"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g011">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g011"
                     alt="Fig 11"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g012">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g012"
                     alt="Fig 12"
                />

            </div>


            <div class="carousel-item lightbox-figure" data-doi="info:doi/10.1371/journal.pone.0147489.g013">

                <img src="/plosone/article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g013"
                     alt="Fig 13"
                />

            </div>
      </div>
    </div>

    <div class="carousel-control">
      <span class="button previous"></span>
      <span class="button next"></span>
    </div>
    <div class="carousel-page-buttons">

    </div>
  </div>
</div>


        <div class="article-text" id="artText">
        



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>CLCA2 is a p53-, p63-inducible transmembrane protein that is frequently downregulated in breast cancer. It is induced during differentiation of human mammary epithelial cells, and its knockdown causes epithelial-to-mesenchymal transition (EMT). To determine how CLCA2 promotes epithelial differentiation, we searched for interactors using membrane dihybrid screening. We discovered a strong interaction with the cell junctional protein EVA1 (Epithelial V-like Antigen 1) and confirmed it by co-immunoprecipitation. Like CLCA2, EVA1 is a type I transmembrane protein that is regulated by p53 and p63. It is thought to mediate homophilic cell-cell adhesion in diverse epithelial tissues. We found that EVA1 is frequently downregulated in breast tumors and breast cancer cell lines, especially those of mesenchymal phenotype. Moreover, knockdown of EVA1 in immortalized human mammary epithelial cells (HMEC) caused EMT, implying that EVA1 is essential for epithelial differentiation. Both EVA1 and CLCA2 co-localized with E-cadherin at cell-cell junctions. The interacting domains were delimited by deletion analysis, revealing the site of interaction to be the transmembrane segment (TMS). The primary sequence of the CLCA2 TMS was found to be conserved in CLCA2 orthologs throughout mammals, suggesting that its interaction with EVA1 co-evolved with the mammary gland. A screen for other junctional interactors revealed that CLCA2 was involved in two different complexes, one with EVA1 and ZO-1, the other with beta catenin. Overexpression of CLCA2 caused downregulation of beta catenin and beta catenin-activated genes. Thus, CLCA2 links a junctional adhesion molecule to cytosolic signaling proteins that modulate proliferation and differentiation. These results may explain how attenuation of CLCA2 causes EMT and why CLCA2 and EVA1 are frequently downregulated in metastatic breast cancer cell lines.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ramena G, Yin Y, Yu Y, Walia V, Elble RC (2016) CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation. PLoS ONE 11(3):
        e0147489.
        doi:10.1371/journal.pone.0147489</p><p><strong>Editor: </strong>Neil A. Hotchin, University of Birmingham, UNITED KINGDOM</p><p><strong>Received: </strong>November 13, 2015; <strong>Accepted: </strong>January 5, 2016; <strong>Published: </strong> March 1, 2016</p><p>This is an open access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the <a href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0</a> public domain dedication.</p><p><strong>Data Availability: </strong>All relevant data are within the paper.</p><p><strong>Funding: </strong>This work was supported by National Cancer Institute 1R15CA151094-01 to RCE. <a href="http://www.cancer.gov/">http://www.cancer.gov/</a>. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Breast cancer relapse is due primarily to metastatic spread that occurs before or during treatment [<a href="#pone.0147489.ref001" class="ref-tip">1</a>]. One of the body’s most potent defenses against metastasis is the anti-proliferative and anti-invasive signaling machinery based at cell-cell junctions. Adherens junctions (AJ) sequester beta catenin, a transcriptional activator of Myc and mesenchymal transcription factor genes that is upregulated in virtually all cancers [<a href="#pone.0147489.ref002" class="ref-tip">2</a>, <a href="#pone.0147489.ref003" class="ref-tip">3</a>]. The loss of epithelial junctional markers during tumor progression is thought to occur by epithelial-to-mesenchymal transition, a process that at once releases cells from anchorage-dependence and confers invasiveness, resistance to chemotherapy, and stem-like properties [<a href="#pone.0147489.ref002" class="ref-tip">2</a>, <a href="#pone.0147489.ref004" class="ref-tip">4</a>, <a href="#pone.0147489.ref005" class="ref-tip">5</a>]. EMT is suppressed by AJ protein E-cadherin, which sequesters beta catenin and inhibits mesenchymal transcription factors [<a href="#pone.0147489.ref006" class="ref-tip">6</a>]. Attenuation of E-cadherin expression is sufficient to drive EMT in mammary epithelium, and E-cadherin is frequently mutated in invasive lobular cancers [<a href="#pone.0147489.ref002" class="ref-tip">2</a>, <a href="#pone.0147489.ref007" class="ref-tip">7</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>The CLCA gene family arose in placozoans, the first multicellular organisms to develop epithelial tissues with organized cell-cell junctions [<a href="#pone.0147489.ref008" class="ref-tip">8</a>]. In mammals CLCAs comprise four subfamilies [<a href="#pone.0147489.ref009" class="ref-tip">9</a>]. They are distinguished by the juxtaposition of metalloprotease and VWA domains and the capacity to self-cleave [<a href="#pone.0147489.ref010" class="ref-tip">10</a>]. CLCA2 is a type I integral transmembrane protein [<a href="#pone.0147489.ref011" class="ref-tip">11</a>]. We recently demonstrated that CLCA2 is a stress-inducible gene, being strongly induced by p53 in response to cell detachment, DNA damage, and other stressors [<a href="#pone.0147489.ref012" class="ref-tip">12</a>]. CLCA2 is frequently downregulated in breast cancers by promoter methylation, and ectopic expression in a breast cancer cell line inhibited tumor formation by tail vein injection and xenograft [<a href="#pone.0147489.ref013" class="ref-tip">13</a>, <a href="#pone.0147489.ref014" class="ref-tip">14</a>]. In vitro, viral transduction inhibited proliferation of HMEC and induced apoptosis or senescence in breast cancer cells, while knockdown reduced mortality in response to the DNA damaging agent doxorubicin [<a href="#pone.0147489.ref012" class="ref-tip">12</a>]. Consistent with an antiproliferative role for CLCA2, a recent study found that it was the most upregulated gene when AP1 oncogenic transcription factor was downregulated and that AP1 components Jun-1 and Fra-1 bound directly to the CLCA2 gene [<a href="#pone.0147489.ref015" class="ref-tip">15</a>]. CLCA2 has also been reported to suppress migration and invasion in breast and colorectal cancer cell lines [<a href="#pone.0147489.ref014" class="ref-tip">14</a>, <a href="#pone.0147489.ref016" class="ref-tip">16</a>].</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>CLCA2 is strongly associated with epithelial differentiation in breast and is downregulated in many breast cancers, most dramatically in the mesenchymal subtype [<a href="#pone.0147489.ref017" class="ref-tip">17</a>]. CLCA2 is upregulated 150-fold when MCF10A HMEC reach confluency, which triggers mesenchymal-to-epithelial transition (MET) in that cell line [<a href="#pone.0147489.ref017" class="ref-tip">17</a>, <a href="#pone.0147489.ref018" class="ref-tip">18</a>]. This association with MET was confirmed in another immortalized HMEC cell line, HMLE, which spontaneously undergoes MET to form cobblestone islands that express E-cadherin and other epithelial markers [<a href="#pone.0147489.ref004" class="ref-tip">4</a>]. CLCA2 was upregulated in the islands [<a href="#pone.0147489.ref017" class="ref-tip">17</a>,<a href="#pone.0147489.ref019" class="ref-tip">19</a>]. Moreover, CLCA2 was downregulated in response to EMT induced by ectopic expression of mesenchymal transcription factors, TGF beta, or cell dilution [<a href="#pone.0147489.ref017" class="ref-tip">17</a>].</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Furthermore, we found that knockdown of CLCA2 by shRNAs provoked EMT in both MCF10A and HMLE, establishing that CLCA2 is a driver of epithelial differentiation rather than a passenger. Indeed, CLCA2 knockdown in HMEC caused focus formation, enhanced invasiveness, and increased mammosphere formation; these changes were accompanied by downregulation of E-cadherin and upregulation of mesenchymal markers [<a href="#pone.0147489.ref017" class="ref-tip">17</a>].</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>To discover how CLCA2 promotes epithelial differentiation, we turned to a surrogate genetic system to search for interacting proteins. We screened cDNA libraries using a two-hybrid system designed for membrane-bound proteins (DualSystems). Although CLCA proteins have been proposed to be accessory proteins for chloride channels [<a href="#pone.0147489.ref020" class="ref-tip">20</a>], the screen did not detect interactions with channels. Instead, one of the strongest interactions proved to be with Epithelial V-like Antigen 1 (EVA1), a Type I transmembrane protein whose ectodomain contains an Ig-like V-domain similar to that of Junctional Adhesion Molecules (JAMs). EVA1 is conserved throughout vertebrates but not beyond (<a href="http://useast.ensembl.org/Multi/GeneTree/Image?gt=ENSGT00640000091161">http://useast.ensembl.org/Multi/GeneTree/Image?gt=ENSGT00640000091161</a>). Like CLCA2, it is regulated by p53, p63, and p73 [<a href="#pone.0147489.ref021" class="ref-tip">21</a>,<a href="#pone.0147489.ref022" class="ref-tip">22</a>,<a href="#pone.0147489.ref023" class="ref-tip">23</a>]. Genes with this regulatory profile are typically involved in terminal differentiation and cell adhesion [<a href="#pone.0147489.ref023" class="ref-tip">23</a>]. EVA1 is thought to mediate homophilic cell-cell adhesion in diverse epithelial tissues, but it has not been studied in breast [<a href="#pone.0147489.ref024" class="ref-tip">24</a>].</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6" class="link-target"></a><p>We show herein that 1) EVA1 is frequently downregulated in breast tumors and breast cancer cell lines, especially those of mesenchymal phenotype; 2) like CLCA2, EVA1 is an epithelial marker that is co-regulated with E-cadherin; 3) knockdown of EVA1 results in rapid EMT in immortalized HMEC; and 4) CLCA2 and EVA1 form a complex with junctional regulatory molecule ZO-1; and 5) that CLCA2 cytoplasmic tail binds to beta catenin. These findings indicate that EVA1 is a cell-cell junction component and that one role of CLCA2 in differentiation is to link cell-cell adhesion to homeostatic signaling.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<h3>Ethics Statement</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The research described herein meets all applicable standards for the ethics of experimentation and research integrity. Human cancer data were derived from public databanks or cell lines that are exempt from IRB requirements. The cell line HMLE was a kind gift from Robert Weinberg (MIT). Its construction from anonymous, commercially available human mammary epithelial cells (Clonetics) has been published [<a href="#pone.0147489.ref004" class="ref-tip">4</a>].</p>


<h3>Cell lines and cell culture</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Breast cell lines MCF-10A, MCF-7, and MDA-MB-231 were obtained from ATCC in 2011 (Manassa, VA, USA) and grown in the media specified. HMLE is a human mammary epithelial line that was immortalized by hTERT [<a href="#pone.0147489.ref004" class="ref-tip">4</a>]. It was a kind gift from Robert Weinberg (MIT, Boston, MA, USA) in 2010 and was grown as described [<a href="#pone.0147489.ref004" class="ref-tip">4</a>]. Knockdown of EVA1 in HMLE and MCF7 was done by lentiviral transduction of small hairpin RNAs (GIPZ, OpenBiosystems). The two most effective out of 5 tested were termed EVA1sh7 and EVA1sh8 (clones #V3LHS_376099 and V3LHS_410864, respectively). Gipz-NC was used as a negative control. MCF-7 and MCF-10A cells were transduced with Myc-tagged EVA1 for localization experiments.</p>


<h3>Plasmid constructions</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Human EVA1 having Myc and Flag-tags at its C-terminus in pCMV6-Entry Vector was purchased from Origene. The flag-tag was removed and EVA1-myc was transferred into pLEX-MCS vector by overlap extension PCR using the forward primer, <span class="monospace">5’ to 3’, GAAGACACCGACTCTACTAGAGGATCCGGTAC CGAGGAGATCTGCCGC</span> and reverse primer, <span class="monospace">5’ to 3’, TTCCACCACACTGGACTAGTCTA GATATCATTTGCTGCCAGATC</span>. Cells transduced with lentivirus were selected with 1 microgram/ml puromycin for a minimum of 2 weeks before performing RNA and protein analysis. For co-expression of EVA1 and CLCA2, EVA1 was inserted into pLenti-GFP-Hygro (Addgene). HEK293 cells were first transduced with pLex-CLCA2, then pLenti-Hygro-EVA1, then selected with 1micrograms/ml puromycin and 50 micrograms /ml hygromycin. CLCA2 deletion mutants were constructed using PCR and other standard molecular methods. Replacement of the CLCA2 ectodomain with Myc-tagged GFP was described previously [<a href="#pone.0147489.ref011" class="ref-tip">11</a>]. Additional variations adding the cytoplasmic tail or substituting the EVA1 TMS (residues 155–180) were constructed by PCR and all were transferred to pLex. A lentivirus expressing CLCA2 from a PGK promoter was constructed by inserting a cassette into the BamHI and KpnI sites of pLKO.1. The cassette encoded puromycin resistance, mCherry, and hCLCA2 linked by T2A and P2A segments.</p>


<h3>Bioinformatics</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Oncomine (Compendia Bioscience, Ann Arbor, MI, USA), Gene Expression Omnibus (NCBI), NextBio, the Cancer Cell Line Encyclopedia (Broad-Novartis Institute), and curated data from The Cancer Genome Atlas were used to analyze the expression patterns of EVA1 in immortalized breast cell lines vs. breast cancer cell lines and breast tissues vs. breast tumor tissues.</p>


<h3>RNA and RT-qPCR analysis</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Cells were grown to confluency except where noted and harvested using Trizol (Invitrogen). RNA was extracted and reverse transcribed as described [<a href="#pone.0147489.ref025" class="ref-tip">25</a>, <a href="#pone.0147489.ref012" class="ref-tip">12</a>]. Expression was quantified by qPCR using an ABI7500 instrument. Primer sequences are available upon request.</p>


<h3>Western blot analysis</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>For protein analysis, cleared NP40 lysates and whole cell SDS lysates were prepared from 10-cm dishes containing cells at 100% confluency as described [<a href="#pone.0147489.ref017" class="ref-tip">17</a>, <a href="#pone.0147489.ref026" class="ref-tip">26</a>]. Protein concentration was measured by BCA assay and 50 micrograms protein was loaded per lane. Immunoprecipitations were performed as described except that the buffer contained 0.5% NP40 [<a href="#pone.0147489.ref011" class="ref-tip">11</a>]. Antibodies for E-cadherin, N-cadherin, beta catenin, phospho-beta catenin, beta four integrin, and fibronectin were all rabbit polyclonal from Cell Signaling; for Myc-tag (clone 9E10.3), from Neomarkers; for FLAG tag (clone M2), Agilent; for actin (125-ACT), from PhosphoSolutions; for GFP, from Abcam; for vimentin, from Millipore. ZO-1 antibody was from PTG. The CLCA2 antibody TVE20 has been described (12). The protein size marker was Dual color (Bio-Rad). Secondary antibodies were labeled with IR680 or IR800 (Licor), and protein expression was quantified on an Odyssey instrument (Licor).</p>


<h3>Immunofluorescence and confocal analysis</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>MCF7 cells stably transduced with EVA1-myc were grown on poly-L-lysine coated coverslips for 8 to 10 days. The cells were fixed with cold methanol, and blocked with blocking buffer (1% FBS and 1% BSA in PBS) overnight at 4C. The cells were then incubated with the same primary antibodies for E-cadherin and ZO-1 used for immunoblots. EVA1 was detected using anti-Myc 9E10. This was followed by secondary antibodies labeled with AlexaFluor 568 or AlexaFluor 488 (Invitrogen) according to manufacturers’ recommendations. Nuclei were stained with 4,6-diamidino-2-phenylindole (DAPI). Coverslips were mounted onto slides using ProLong Gold mounting medium (Invitrogen). The cells were visualized under a fluorescent compound microscope prior to confocal laser-scanning microscopic analysis (Leica LAS-TF). The cells were scanned for Z-stacks from basal to apical side. Images were collected from XYZ-axes with each section thickness 1μm. Orthogonal images were obtained using the Leica software in XY and XZ planes to analyze the spatial localization of junctional proteins. Transduced HEK-293 cells were grown to superconfluency and treated similarly. We found that high cell density was necessary for junctional maturation and apical localization of ZO-1 in this cell line. MCF10A cells were stained with anti-CLCA2 antibody (Sigma Prestige, #HPA47192) and examined using an inverted microscope (Olympus IX71).</p>


<h3>Statistics</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Each experiment was repeated three times and Student’s t-test was performed to determine the statistical significance of the results. P values less than 0.05 were considered significant.</p>

</div>

<div id="section3" class="section toc-section"><a id="sec011" name="sec011" data-toc="sec011" class="link-target" title="Results"></a><h2>Results</h2>
<h3>Interaction of CLCA2 and EVA1</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>To determine how CLCA2 promotes MET, we conducted a two-hybrid screen using a split-ubiquitin system designed for transmembrane proteins, wherein bait and prey are expressed in the yeast endoplasmic reticulum (DualSystems Biotech, Basel, Switzerland). Putative interactors from the yeast screen were further tested by co-transfection into 293T cells followed by co-immunoprecipitation (co-IP) assays. Candidate proteins were labeled with C-terminal FLAG tags. We found that EVA1, which ran as a cluster of three diffuse bands, could be precipitated by CLCA2 antibody from co-transfected cells but not from cells transfected with EVA1 alone (<a href="#pone-0147489-g001">Fig 1A</a>). Similarly, CLCA2 was precipitated with FLAG antibody from co-transfected cells but not from cells transfected with CLCA2 alone. A second two-hybrid interactor, TMEM55b, did not interact by co-IP and thus served as an additional negative control (<a href="#pone-0147489-g001">Fig 1B</a>).</p>
<a class="link-target" id="pone-0147489-g001" name="pone-0147489-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g001" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g001">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g001">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g001">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g001">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Co-immunoprecipitation assay of two-hybrid interactors.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>Flag-tagged candidate cDNAs were transfected into HEK293T cells with or without CLCA2 cDNA and lysates were subjected to co-immunoprecipitation and immunoblot with antibodies specific for FLAG or CLCA2. The CLCA2 antibody was TVE20. A, EVA1 was precipitated by CLCA2 antibody only from cells transfected (TF) with both. X, nonspecific bands. B, TMEM55b failed to co-precipitate. E, EVA1; h2, human CLCA2; 55b, TMEM55b. Vertical lines indicate omitted lanes. Sizes marked in kilodaltons.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g001">
              http://dx.doi.org/10.1371/journal.pone.0147489.g001</a></p></div>

<h3>CCA2 binds EVA1 via its transmembrane segment</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>To gain insight into the purpose of the interaction, we asked which domain of CLCA2 interacted with EVA1. Testing of deletions encompassing the region C-terminal to the cleavage site, the transmembrane segment (TMS), or the cytoplasmic tail (CT) suggested that the TMS was responsible (<a href="#pone-0147489-g002">Fig 2B and 2C</a>). Alternatively, removing the TMS might have merely disrupted the plasma membrane localization of the protein, preventing the interaction. To test this possibility, the ectodomain was replaced with GFP (<a href="#pone-0147489-g003">Fig 3A and 3B</a>). A strong co-IP was still observed. Removal of the CT had no effect on the interaction, and EVA1 failed to interact when the CT alone was fused to GFP, ruling out a role for that domain (<a href="#pone-0147489-g003">Fig 3C and 3D</a>). These results confirm that EVA1 interacts with the TMS of CLCA2. To rule out the possibility that EVA1 might interact indiscriminately with any TMS, EVA1 and integrin beta four were co-transfected and immunoprecipitated. No interaction was seen (<a href="#pone-0147489-g003">Fig 3E</a>).</p>
<a class="link-target" id="pone-0147489-g002" name="pone-0147489-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g002" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g002">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g002">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g002">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g002">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Identification of the CLCA2 domain that interacts with EVA1.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>A. constructs used to identify the CLCA2 domain that interacts with EVA1. CLCA2 interaction with EVA1 is denoted by + or - after each construct. B-D, immunoprecipitation from co-transfected cells indicates requirement for the transmembrane segment (TMS). EVA1 was detected with Flag antibody and CLCA2, with TVE20.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g002">
              http://dx.doi.org/10.1371/journal.pone.0147489.g002</a></p></div><a class="link-target" id="pone-0147489-g003" name="pone-0147489-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g003" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g003">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g003">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g003">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g003">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Confirmation of the TMS as the interacting domain.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>A, substitution of GFP for the CLCA2 ectodomain with CLCA2 C-terminal domains appended. CLCA2 interaction with EVA1 is denoted by + or - after each construct. B- D, immunoprecipitation from cells co-transfected with constructs and EVA1 indicates that CLCA2 TMS is required but not cytoplasmic tail. E, immunoprecipitation from cells co-transfected with EVA1 and integrin beta four. F, immunoprecipitation from cells co-transfected with ssGFP tagged with the EVA1 TMS. ssGFP by itself failed to interact with either CLCA2 or EVA1 (not shown). E, EVA1. G, GFP. Antibodies: EVA1, Flag; GFP constructs, GFP; CLCA2, TVE20.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g003">
              http://dx.doi.org/10.1371/journal.pone.0147489.g003</a></p></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>As the interaction with EVA1 must occur within the membrane bilayer, the TMS of EVA1 was considered most likely to interact with that of CLCA2. Accordingly, appending the EVA1 TMS to ssGFP allowed it to co-IP with CLCA2 (<a href="#pone-0147489-g003">Fig 3F</a>). Alignment of the TMS segments of mammalian CLCA2 homologs revealed that the sequence of the TMS is highly conserved, especially the lysine, cysteine, and three glycine residues, suggesting a function beyond anchoring the protein in the lipid bilayer (<a href="#pone-0147489-g004">Fig 4</a>). The other human homologs, CLCA1 and CLCA4, lack this sequence, suggesting that interaction with EVA1 is intrinsic to CLCA2.</p>
<a class="link-target" id="pone-0147489-g004" name="pone-0147489-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g004" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g004">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g004">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g004">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g004">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Conservation of CLCA2 TMS amino acid sequence in mammals.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1" class="link-target"></a><p>Six positions were conserved in all mammals tested and five diverged in only one species. Dissimilar substitutions are boxed. Data from UNIPROT were aligned by ClustalW, JALview.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g004">
              http://dx.doi.org/10.1371/journal.pone.0147489.g004</a></p></div>

<h3>EVA1 is an epithelial marker that is downregulated in breast cancer EMT</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>By searching public databases, we found that EVA1 was frequently downregulated in breast cancer cell lines with mesenchymal expression profiles compared to cell lines that retained epithelial markers, such as some luminal and basal (<a href="#pone-0147489-g005">Fig 5A</a>). We obtained similar results using the Cancer Cell Line Encyclopedia dataset [<a href="#pone.0147489.ref003" class="ref-tip">3</a>] and further observed that EVA1 expression level correlated with that of junctional marker E-cadherin (<a href="#pone-0147489-g005">Fig 5B</a>). This was confirmed by RT-qPCR with representative cell lines (<a href="#pone-0147489-g005">Fig 5C</a>).</p>
<a class="link-target" id="pone-0147489-g005" name="pone-0147489-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g005" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g005">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g005">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g005">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g005">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 5. </span> EVA1 downregulation in breast cancer.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>A, EVA1 gene expression profiles extracted from a study of breast cell lines [<a href="#pone.0147489.ref059" class="ref-tip">59</a>] using GEO. B, heat-maps comparing EVA1 with E-cadherin expression in breast cell lines using data from the Cancer Cell Line Encyclopedia (3). Green indicates lowest expression and red indicates highest. C, expression profiles by qPCR showing downregulation of EVA1 and E-cadherin in the mesenchymal cell line MDA-MB-231 relative to luminal MCF7. Values were normalized to levels in immortalized HMEC, HMLE. p &lt;0.05 for all pairwise comparisons. D, breast cancer expression profiles extracted from The Cancer Genome Atlas (TCGA1) databank using Oncomine. The log2 median-centered ratios are depicted in box-and-whisker plots. Dots represent maximum and minimum outliers from the main dataset. For each plot, the following pathological subtypes were evaluated separately. 0, normal tissue (61); 1, apocrine carcinoma (1); 2, large cell neuroendocrine (1); 3, ductal carcinoma (1); 4, intraductal cribriform adenocarcinoma (3); 5, invasive carcinoma (76); 6, invasion cribriform carcinoma (1); 7, invasive ductal carcinoma (395); 8, invasive ductal and lobular carcinoma (3); 9, invasive lobular breast carcinoma (36); 10, invasive papillary breast carcinoma (1); 11, metaplastic breast carcinoma (1); 12, mixed lobular and ductal breast carcinoma (7); 13, mucinous breast carcinoma (4); 14, papillary carcinoma (1); 15, pleomorphic carcinoma (1). Number in parentheses indicates sample size for each category.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g005">
              http://dx.doi.org/10.1371/journal.pone.0147489.g005</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>To determine EVA1 status in breast tumors, we queried the TCGA breast cancer dataset using Oncomine (<a href="#pone-0147489-g005">Fig 5D</a>). We found that EVA1 expression was often downregulated in breast tumors (lanes 1–15) relative to normal breast tissue (lane 0). As above, tumor types with high median expression of EVA1 also had high expression of E-cadherin (data not shown). It should also be noted that EVA1 is expressed in some lymphocytes [<a href="#pone.0147489.ref027" class="ref-tip">27</a>] and arterial endothelial cells [<a href="#pone.0147489.ref028" class="ref-tip">28</a>], so that immune infiltration or tumor vascularization could inflate apparent expression in tumor samples.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3" class="link-target"></a><p>These results paralleled the behavior of CLCA2 and suggested that EVA1 might also be a marker of epithelial phenotype. To test this, we used Oncomine and NextBio to consult transcriptional profiles from cell culture systems that model mammary epithelial differentiation. In the first study, primary human mammary epithelial cells were cultured in three-dimensional Matrigel to promote epithelial differentiation or grown as mammospheres to promote stem-like properties [<a href="#pone.0147489.ref029" class="ref-tip">29</a>]. Expression of EVA1 and CLCA2 was seven to eightfold higher in the Matrigel population (<a href="#pone-0147489-t001">Table 1</a>). In the second study, the immortalized mammary epithelial cell line MCF10A was cultured in monolayer on plastic or on permeable membranes that support normal apico-basal polarization, barrier formation, and other aspects of differentiation [<a href="#pone.0147489.ref030" class="ref-tip">30</a>]. EVA1 expression was threefold higher and CLCA2 was seven times higher in the differentiated population. On the other hand, the genes were downregulated in parallel when EMT was induced by TGF beta, expression of mesenchymal transcription factors, knockdown of E-cadherin [<a href="#pone.0147489.ref031" class="ref-tip">31</a>], or ectopic expression of miR221/222 [<a href="#pone.0147489.ref032" class="ref-tip">32</a>].</p>
<a class="link-target" id="pone-0147489-t001" name="pone-0147489-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.t001" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.t001">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.t001">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.t001">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.t001">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Table 1. </span> Association with epithelial differentiation.</div><p class="caption_target"></p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.t001">
              http://dx.doi.org/10.1371/journal.pone.0147489.t001</a></p></div><a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4" class="link-target"></a><p>We confirmed the association of EVA1 with epithelial phenotype using the mammary epithelial cell line HMLE, which undergoes spontaneous MET with increasing cell density, forming cobblestone-like islands with high expression of E-cadherin and other junctional markers [<a href="#pone.0147489.ref004" class="ref-tip">4</a>]. The islands can be separated from surrounding mesenchymal-like cells by differential trypsinization [<a href="#pone.0147489.ref019" class="ref-tip">19</a>]. Transcriptional profiling of the fractions by RT-qPCR revealed that EVA1 correlates directly with E-cadherin but inversely with mesenchymal markers (<a href="#pone-0147489-g006">Fig 6</a>). These results indicate that EVA1 is an epithelial marker whose expression is lost upon EMT.</p>
<a class="link-target" id="pone-0147489-g006" name="pone-0147489-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g006" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g006">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g006">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g006">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g006">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 6. </span> Expression of EVA1 correlates with epithelial differentiation.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1" class="link-target"></a><p>HMLE cells were separated into epithelioid and mesenchymal subpopulations by differential trypsinization and subjected to RT-qPCR. The transcriptional profile reveals that EVA1 is highest in the most trypsin-resistant fraction, correlating with E-cadherin and CLCA2. The 5min fraction was normalized as 1. P&lt;0.01 for all comparisons between 5min and 20min samples.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g006">
              http://dx.doi.org/10.1371/journal.pone.0147489.g006</a></p></div>

<h3>EVA1 knockdown triggers EMT</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Cell junctional adhesion proteins such as E-cadherin mediate signaling pathways that maintain epithelial differentiation, and their downregulation leads to EMT and tumorigenesis [<a href="#pone.0147489.ref002" class="ref-tip">2</a>,<a href="#pone.0147489.ref004" class="ref-tip">4</a>]. We supposed that EVA1, as a junctional protein, might also promote epithelial differentiation and that CLCA2 might promote epithelial integrity by regulating EVA1 localization or activity. To test this hypothesis, we knocked down EVA1 in HMLE using two lentiviral constructs, sh7 and sh8 (Gipz, OpenBiosystems). Both caused a change to mesenchymal morphology, loss of cell clustering at low density, and focus formation at high density (<a href="#pone-0147489-g007">Fig 7A</a>). Transcriptional and protein profiling confirmed EVA1 knockdown and a shift from epithelial to mesenchymal phenotype (<a href="#pone-0147489-g007">Fig 7B and 7C</a>). These results revealed that EVA1 is required for epithelial phenotype in vitro.</p>
<a class="link-target" id="pone-0147489-g007" name="pone-0147489-g007"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g007" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g007">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g007">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g007">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g007">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 7. </span> Knockdown of EVA1 induces EMT.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>HMLE cells were lentivirally transduced with EVA1 shRNAs sh7 and sh8 or non-silencing control GipzNC. <strong>A</strong>, phase-contrast (top) and fluorescence micrographs showing changes in cell behavior after knockdown. Top was stained with crystal violet, bottom shows GFP. Scale bar, 200 microns. <strong>B</strong>, immunoblots of whole cell lysates probed for EMT marker proteins. <strong>C</strong>, transcriptional profile showing attenuation of EVA1 and EMT in knockdown cell lines. Values were normalized to HMLE parent. P&lt;0.01 for each pairwise comparison between GipzNC and EVAsh7 or EVAsh8.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g007">
              http://dx.doi.org/10.1371/journal.pone.0147489.g007</a></p></div>

<h3>Co-Immunolocalization of EVA1 and CLCA2</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>The structure of EVA1 resembles that of Junctional Adhesion Molecules (JAMs) which reside at tight junctions. To determine its localization, we transduced EVA1 into a breast cancer cell line known to form tight junctions, MCF7, and performed immunofluorescence and confocal microscopy. As expected, the cells showed apicolateral localization of the tight junction marker ZO-1 and lateral localization of E-cadherin (<a href="#pone-0147489-g008">Fig 8A</a>). Surprisingly however, EVA1 was found throughout the lateral junction like E-cadherin rather than being apical (<a href="#pone-0147489-g008">Fig 8B and 8C</a>).</p>
<a class="link-target" id="pone-0147489-g008" name="pone-0147489-g008"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g008" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g008">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g008">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g008">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g008">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 8. </span> EVA1 localizes throughout lateral interfaces of cell-cell junctions.</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1" class="link-target"></a><p>Confocal micrographs of MCF7-EVA1 cells grown to high density and probed for A, E-cadherin and ZO-1; B, E-cadherin and EVA1; or C, ZO-1 and EVA-1. In A and B, nuclei were stained with DAPI (blue). To the right of each XY image is a YZ image showing that EVA1localization resembles that of E-cadherin rather than ZO-1. Scale bar, 50 microns.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g008">
              http://dx.doi.org/10.1371/journal.pone.0147489.g008</a></p></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>To determine CLCA2 localization, we initially used HEK293 cells because MCF7 cells do not tolerate its constitutive expression [<a href="#pone.0147489.ref012" class="ref-tip">12</a>]. Unlike CLCA1 and CLCA4, its C-terminus contains a conserved match for the AP1-mediated basolateral sorting signal, D/ExxxLL [<a href="#pone.0147489.ref033" class="ref-tip">33</a>]. Immunolocalization of HEK293 cells transduced with FLAG-tagged CLCA2 confirmed that CLCA2 is found both in the basal membrane and lateral junctions (<a href="#pone-0147489-g009">Fig 9A</a>). The junctional labeling was observed only in cells that had been confluent for several days and started to form multiple cell layers. In non-confluent cells, CLCA2 was found mostly in cytoplasmic vesicles.</p>
<a class="link-target" id="pone-0147489-g009" name="pone-0147489-g009"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g009"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g009" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g009"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g009" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g009">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g009">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g009">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g009">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g009">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g009">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g009">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g009">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 9. </span> CLCA2 localizes to basal and lateral interfaces of cell junctions.</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig2.caption1.p1" name="article1.body1.sec3.sec5.fig2.caption1.p1" class="link-target"></a><p><strong>A</strong>, confocal images of HEK293-CLCA2 cells grown to superconfluency and probed for CLCA2 or ZO-1. Top left, a basal plane from the Z-stack; top right, a central plane. Lower right, YZ image shows ZO-1 at apex of lateral junction, while CLCA2 occurs throughout junction. <strong>B</strong>, detection of endogenous CLCA2 in MCF10A (left) but not in cells expressing CLCA2 shRNA (right). Scale bars, 50 microns.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g009">
              http://dx.doi.org/10.1371/journal.pone.0147489.g009</a></p></div><a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3" class="link-target"></a><p>To determine localization of endogenous CLCA2 in MEC, we used a novel antibody to perform IF on confluent MCF10A and knockdown controls (<a href="#pone-0147489-g009">Fig 9B</a>). A strong signal was seen at cell-cell junctions in wildtype but not knockdown cells.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4" class="link-target"></a><p>To determine whether CLCA2 and EVA1 co-localize, HEK293 cells were co-transduced with both EVA1 and CLCA2 and grown to high cell density. We observed overlapping junctional localization, along with cytoplasmic expression (<a href="#pone-0147489-g010">Fig 10</a>, arrows).</p>
<a class="link-target" id="pone-0147489-g010" name="pone-0147489-g010"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g010"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g010" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g010"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g010" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g010">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g010">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g010">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g010">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g010">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g010">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g010">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g010">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 10. </span> CLCA2 and EVA1 co-localize at cell-cell junctions.</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig3.caption1.p1" name="article1.body1.sec3.sec5.fig3.caption1.p1" class="link-target"></a><p>Confocal images of HEK293 cells stably expressing CLCA2-flag and EVA1-myc and stained for EVA1 (green), CLCA2 (red), and nuclei (blue). To the right is a YZ image. Arrows denote junctional staining. Scale bar, 50 microns.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g010">
              http://dx.doi.org/10.1371/journal.pone.0147489.g010</a></p></div>

<h3>CLCA2 and EVA1 interact with junctional signaling molecules</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>To determine how these proteins affect junctional signaling, we tested whether EVA1 or CLCA2 interacted with known components of TJ and AJ by co-IP from cells expressing either protein. Most potential interactors were negative, including E-cadherin, occludin, Zonab, and claudins (data not shown). However, CLCA2 interacted with signaling molecules from both TJ and AJ, ZO-1 and beta catenin (<a href="#pone-0147489-g011">Fig 11A and 11B</a>). Deletion of the cytoplasmic tail of CLCA2 abolished the interaction (<a href="#pone-0147489-g011">Fig 11C</a>). ZO-1 is an adapter protein that links TJ and AJ adhesion molecules to the cytoskeleton and is required for maturation of AJ [<a href="#pone.0147489.ref034" class="ref-tip">34</a>]. This suggests that the CLCA2-EVA1 interaction links cell-cell adhesion by EVA1 to both TJ and AJ signaling pathways.</p>
<a class="link-target" id="pone-0147489-g011" name="pone-0147489-g011"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g011"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g011" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g011"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g011" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g011">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g011">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g011">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g011">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g011">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g011">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g011">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g011">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 11. </span> CLCA2 interaction with ZO-1 and β-catenin.</div><p class="caption_target"><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong> and <strong>B</strong>, HEK293 cells with or without stable CLCA2 expression were grown to superconfluency, and lysates were subjected to immunoprecipitation and immunoblot with antibodies specific for either CLCA2 (h2), ZO-1(Z) or β-catenin (β). <strong>C</strong>, cells expressing CLCA2 lacking the cytoplasmic tail were analyzed identically; no interactions were observed. <strong>D</strong>, failure of transduced EVA1 to co-IP with ZO-1 or beta catenin. <strong>E</strong>, co-IP of EVA1 with ZO-1 when CLCA2 was co-expressed. Except in <strong>C</strong>, CLCA2 contained a FLAG tag and EVA1 contained a Myc tag. In <strong>C</strong>, TVE20 antibody was used.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g011">
              http://dx.doi.org/10.1371/journal.pone.0147489.g011</a></p></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2" class="link-target"></a><p>To determine whether CLCA2 allowed EVA1 to interact with ZO-1 and beta catenin, cells were transduced with both CLCA2 and EVA1, and co-IP studies were performed. We found that EVA1 co-purified with ZO-1 only in the presence of CLCA2 (<a href="#pone-0147489-g011">Fig 11D and 11E</a>). However, beta catenin did not co-IP with EVA1 (data not shown), suggesting that CLCA2 forms two independent complexes, one with EVA1 and ZO-1, the other with beta catenin and perhaps other adherens junction proteins.</p>


<h3>CLCA2 downregulates beta-catenin target genes</h3>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1" class="link-target"></a><p>To determine whether CLCA2 affected beta catenin positively or negatively, we constructed a lentivirus expressing CLCA2 from a weak promoter to reduce toxicity and transduced it into MCF7 cells. Expression profiling of selected cells revealed downregulation of beta catenin target genes such as Cox2, Myc, and mesenchymal transcription factors such as Twist1 and Snail1 (<a href="#pone-0147489-g012">Fig 12</a>). These results suggest that CLCA2 is a negative regulator of beta catenin, the principal driver of EMT and effector of Wnt signaling. Furthermore, they may explain why ectopic expression of CLCA2 inhibits proliferation or migration in many cell types and knockdown triggers EMT in HMEC [<a href="#pone.0147489.ref014" class="ref-tip">14</a>, <a href="#pone.0147489.ref016" class="ref-tip">16</a>, <a href="#pone.0147489.ref017" class="ref-tip">17</a>]. In contrast, transduction of EVA1 alone into MCF10A or MDA-MB231 had no apparent effect on proliferation or migration (data not shown). The former has high expression of both genes while the latter lacks both. The lack of effect of EVA1 overexpression in either cell line is consistent with a scenario in which EVA1 promotes differentiation through CLCA2, perhaps by regulating its localization or activity. Endogenous EVA1 already fulfills that role in MCF10A, so further EVA1 overexpression has no effect. On the other hand, the lack of CLCA2 in MDA-MB231 deprives EVA1 of the tool by which it promotes MET, so no differentiative effect is observed.</p>
<a class="link-target" id="pone-0147489-g012" name="pone-0147489-g012"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g012"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g012" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g012"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g012" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g012">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g012">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g012">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g012">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g012">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g012">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g012">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g012">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 12. </span> CLCA2 downregulates beta catenin-regulated genes.</div><p class="caption_target"><a id="article1.body1.sec3.sec7.fig1.caption1.p1" name="article1.body1.sec3.sec7.fig1.caption1.p1" class="link-target"></a><p>Transcriptional profile of beta catenin-client genes (boxed) in MCF7 cells engineered to moderately express CLCA2. Values are normalized to vector-transduced control (NC). P&lt;0.05 for comparisons between MCF7-CLCA2 and control within the boxed set.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g012">
              http://dx.doi.org/10.1371/journal.pone.0147489.g012</a></p></div>
</div>

<div id="section4" class="section toc-section"><a id="sec019" name="sec019" data-toc="sec019" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Stable cell-cell junctions are required for maintaining epithelial homeostasis, a primary defense against invasion and metastasis. Attenuation of E-cadherin or ZO-1 causes not only loss of junctional anchoring but triggers global programmatic changes that abet invasion and metastasis [<a href="#pone.0147489.ref004" class="ref-tip">4</a>, <a href="#pone.0147489.ref034" class="ref-tip">34</a>]. Having observed similar changes in response to knockdown of CLCA2, we sought to understand the mechanism by using surrogate genetics. These studies revealed interaction with cell-cell adhesion protein EVA1 and unexpected interactions with the same signaling interface as E-cadherin, while no direct interaction with E-cadherin was found.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>EVA1 was originally characterized as an epithelial cell-cell adhesion protein with a structure similar to that of tight-junctional JAMs [<a href="#pone.0147489.ref024" class="ref-tip">24</a>]. However, it is best known from studies of the choroid plexus, which forms the barrier between the blood and cerebrospinal fluid [<a href="#pone.0147489.ref027" class="ref-tip">27</a>, <a href="#pone.0147489.ref035" class="ref-tip">35</a>]. There, EVA1 is expressed by epithelial cells whose tight junctions restrict the passage of T cells into the CNS. Rag-/- mice lacking T-cells were shown to have increased permeability at the choroid plexus, but treating these mice with an activating antibody that recognizes EVA1 ectodomain increased expression and junctional localization of E-cadherin and reduced permeability of the choroid plexus. The authors proposed that ligation of EVA1 by receptors on T cells reinforces cell-cell junctions and restricts subsequent T cell entry. Thus, the passage of one T cell closes the door to followers. In support of this model, the same lab found that an EVA1 knockout mouse could no longer exclude T cells from the brain or B cells from the CSF [<a href="#pone.0147489.ref027" class="ref-tip">27</a>]. In addition, they showed that treatment of HEK cells overexpressing EVA1 with the activating antibody induced intracellular Ca<sup>+2</sup> which is important for E-cadherin function at cell-cell junctions [<a href="#pone.0147489.ref035" class="ref-tip">35</a>].</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>We found here that, like CLCA2 and E-cadherin, EVA1 is also an epithelial differentiation marker that is required for maintaining differentiation of mammary epithelial cells. EVA1 and CLCA2 both localized throughout the lateral junctions, similar to E-cadherin. To understand how these two junctional proteins mediated differentiation, we screened for TJ and AJ molecule interactions either with CLCA2 or EVA1 or both together. Our results suggest that CLCA2 mediates epithelial differentiation through two different complexes. The first complex contains EVA1 bound to CLCA2 TMS and ZO-1 bound to the cytoplasmic tail of CLCA2 (<a href="#pone-0147489-g013">Fig 13A</a>). In the second complex, beta catenin is bound to the cytoplasmic tail of CLCA2; EVA1 is absent from this complex.</p>
<a class="link-target" id="pone-0147489-g013" name="pone-0147489-g013"></a><div class="figure" data-doi="10.1371/journal.pone.0147489.g013"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0147489.g013" data-doi="info:doi/10.1371/journal.pone.0147489" data-uri="info:doi/10.1371/journal.pone.0147489.g013"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0147489.g013" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g013">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0147489.g013">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g013">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0147489.g013">
                larger image
                <span class="file-size" data-size="large" data-doi="info:doi/10.1371/journal.pone.0147489.g013">
                  ()
                </span></a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g013">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0147489.g013">
                original image
                <span class="file-size" data-size="original" data-doi="info:doi/10.1371/journal.pone.0147489.g013">
                  ()
                </span></a></li></ul></div><div class="figcaption"><span>Fig 13. </span> Proposed model for action of CLCA2 and EVA1 at cell-cell junctions.</div><p class="caption_target"><a id="article1.body1.sec4.fig1.caption1.p1" name="article1.body1.sec4.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong>, CLCA2 forms two complexes, one with beta catenin that is independent of EVA1, and the other containing ZO-1 and EVA1, when present. Homophilic ligation of EVA1 may transduce a signal to the cytoskeleton through CLCA2 interaction with ZO-1. CLCA2 is shown as a monomer for simplicity. <strong>B</strong>, pathways by which CLCA2 may promote epithelial differentiation and homeostasis. See text for details.</p>
</p><p class="caption_object"><a href="http://dx.doi.org/10.1371/journal.pone.0147489.g013">
              http://dx.doi.org/10.1371/journal.pone.0147489.g013</a></p></div><a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Both complexes have regulatory implications (<a href="#pone-0147489-g013">Fig 13B</a>). ZO-1 is important for tight junction formation as well as formation of belt-like adherens junctions during epithelial differentiation [<a href="#pone.0147489.ref034" class="ref-tip">34</a>]. ZO-1 is a membrane-associated guanylate kinase with multiple protein-protein interaction motifs and an actin binding region [<a href="#pone.0147489.ref036" class="ref-tip">36</a>]. Tsukita found that loss of ZO-1 impaired the activation of Rac1, which is required for belt-like AJ formation during epithelial polarization [<a href="#pone.0147489.ref034" class="ref-tip">34</a>]. These results suggest that the complex of CLCA2 and EVA1 with ZO-1 may promote maturation of AJ and epithelial polarization.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>On the other hand, beta catenin is a signaling effector of Wnt and growth factor receptors and is stabilized by the AKT pathway [<a href="#pone.0147489.ref037" class="ref-tip">37</a>]. Its transcriptional targets include effectors of proliferation and mesenchymal transcription factors [<a href="#pone.0147489.ref038" class="ref-tip">38</a>–<a href="#pone.0147489.ref041" class="ref-tip">41</a>]. We found that ectopic expression of CLCA2 destabilizes beta catenin and downregulates these genes. Beta catenin stability is known to be regulated by a cytoplasmic complex containing GSK3, axin, and APC; disruption of this complex is linked to progression of cancers of the colon and other organs [<a href="#pone.0147489.ref042" class="ref-tip">42</a>]. It is possible that CLCA2 cytoplasmic tail participates in this complex, perhaps tethering it to the plasma membrane. These results further suggest that CLCA2 may be useful in damping beta catenin activity in Wnt-dependent cancers.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>An interesting teleological question is why cells need CLCA2 to link cell-cell adhesion to beta catenin signaling when E-cadherin already performs this function. E-cadherin appears to be one of the initial interactors when cells meet [<a href="#pone.0147489.ref043" class="ref-tip">43</a>–<a href="#pone.0147489.ref045" class="ref-tip">45</a>]. However, these contacts between membranes are sporadic and do not engage the cytoskeleton [<a href="#pone.0147489.ref046" class="ref-tip">46</a>–<a href="#pone.0147489.ref048" class="ref-tip">48</a>]. Maturation of these contacts into an adhesion belt linked to cytoskeletal actin requires the release of intracellular calcium, which promotes association of ZO-1, AJ, and desmosomes with the cytoskeleton and mobilizes occludin to TJ [<a href="#pone.0147489.ref049" class="ref-tip">49</a>, <a href="#pone.0147489.ref050" class="ref-tip">50</a>]. It is possible that EVA1-EVA1 contacts occur after E-cadherin initial contacts, and EVA1, with or without CLCA2, is responsible for the burst of intracellular calcium required for maturation of adherens junctions (<a href="#pone-0147489-g013">Fig 13B</a>). CLCA2 also differs in having an enzymatic ectodomain that might inhibit Wnt or other extracellular signaling factors. In this light, in airway epithelial cells, the CLCA1 diffusible ectodomain was shown to enhance the activity of Ano1, activate mucus secretion and transdifferentiation, and activate macrophages [<a href="#pone.0147489.ref051" class="ref-tip">51</a>, <a href="#pone.0147489.ref052" class="ref-tip">52</a>, <a href="#pone.0147489.ref053" class="ref-tip">53</a>]. It is tempting to speculate that CLCA2 performs similar functions in mammary epithelium. These possibilities remain to be tested.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7" class="link-target"></a><p>The subcellular localization of CLCA2 has never been investigated in HMEC because of a lack of specific antibodies and low protein expression. Here we first used epitope tags to document basolateral localization of ectopic CLCA2 in HEK293 cells and then took advantage of a recently developed antibody to show lateral localization of endogenous CLCA2 in MCF10A. Accordingly, others have reported junctional localization in esophageal squamous epithelium [<a href="#pone.0147489.ref054" class="ref-tip">54</a>] and basal localization in hemidesmosomes of corneal epithelium [<a href="#pone.0147489.ref055" class="ref-tip">55</a>]. This pattern may be significant because basal and lateral adhesion signaling are coordinated through multiple pathways to promote or inhibit cell migration [<a href="#pone.0147489.ref056" class="ref-tip">56</a>], suggesting a possible role for CLCA2 in cross-talk between AJ and hemidesmosomes. In addition to membrane expression, both CLCA2 and EVA1 were present in intracellular vesicles. Thus, we cannot rule out additional, cytoplasmic functions for either protein.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8" class="link-target"></a><p>What is the functional significance of the interaction between these proteins? Our finding that the EVA1-CLCA2 interaction occurs between the transmembrane segments suggests that the interaction links the activities of the two proteins. Such allosteric interactions may affect localization, dimerization, or activity of the protein partner [<a href="#pone.0147489.ref057" class="ref-tip">57</a>]. Thus, CLCA2 might promote cell-cell junction formation by localizing EVA1, or EVA1 may direct CLCA2 to the cell-cell junction. Another possibility is that CLCA2 could regulate EVA1 dimerization or ligation to a partner on an opposing cell; or that EVA1 may regulate CLCA2 dimerization and cleavage. Heterodimer formation might prevent CLCA2-CLCA2 interaction and cleavage. Further studies are needed to address these possibilities. Whatever its purpose, the conservation of the CLCA2 TMS primary sequence in mammals suggests that this interaction has played an important role in the evolution of mammalian epithelia.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9" class="link-target"></a><p>The EVA1-CLCA2 complex may also play an important role in paracellular trafficking of immune cells. One of the functions of epithelial cell-cell junctions in all organs is to regulate the trafficking of immune cells into the tissues [<a href="#pone.0147489.ref058" class="ref-tip">58</a>]. From the work of Carrithers, EVA1 seems designed to interact with T cells at cell-cell junctions. It is possible that CLCA2 is also involved in EVA1 barrier function at the choroid plexus and that the complex performs a similar barrier or immune filtering function in breast and other tissues. It is tempting to speculate that the metalloprotease activity of CLCA2 may be used by EVA1 to regulate the passage of such cells into tissues in the course of routine immunosurveillance or in response to stress. Thus, in addition to the effects on epithelial differentiation, the loss of EVA1 and/or CLCA2 in breast epithelium might impair immunosurveillance that normally suppresses cancer emergence.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec020" name="sec020" data-toc="sec020" class="link-target" title="Conclusion"></a><h2>Conclusion</h2><a id="article1.body1.sec5.p1" name="article1.body1.sec5.p1" class="link-target"></a><p>By surrogate genetics we have identified an important cell-cell junctional signaling mechanism that remained undetected until now because it does not appear to exist in non-mammalian genetic systems. The data fill a gap in our understanding of how junctional signaling suppresses the activity of beta catenin which runs amok in invasive cancers. Future studies to identify other binding partners at cell-cell junctions and functional studies of the role of CLCA2 metalloprotease activity on the organization of lateral junctions may reveal new tumor suppressive pathways that can be exploited in cancer prevention or therapy.</p>
</div>





<div class="toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We are grateful to technical specialist Wei Du for expert assistance with confocal microscopy.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: GR RCE. Performed the experiments: GR Y. Yin Y. Yu VW. Analyzed the data: GR Y. Yin Y. Yu VW RCE. Wrote the paper: GR RCE.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0147489.ref001" id="pone.0147489.ref001" class="link-target"></a>Creighton CJ, Li X, Landis M, Dixon JM, Neumeister VM, Sjolund A, et al. Residual breast cancers after conventional therapy display mesenchymal as well as tumor-initiating features. <em>Proc Natl Acad Sci</em> USA 2009; 106(33):13820–13825.  doi: 10.1073/pnas.0905718106.  pmid:19666588 <ul class="reflinks" data-doi="10.1073/pnas.0905718106"><li><a href="http://dx.doi.org/10.1073/pnas.0905718106" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Creighton%5Bauthor%5D+AND+Residual+breast+cancers+after+conventional+therapy+display+mesenchymal+as+well+as+tumor-initiating+features" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Residual+breast+cancers+after+conventional+therapy+display+mesenchymal+as+well+as+tumor-initiating+features&amp;author=Creighton&amp;publication_year=2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0147489.ref002" id="pone.0147489.ref002" class="link-target"></a>Onder TT, Gupta PB, Mani SA, Yang J, Lander ES, Weinberg RA. Loss of E-cadherin promotes metastasis via multiple downstream transcriptional pathways. Cancer Res 2008; 68(10):3645–3654.  doi: 10.1158/0008-5472.CAN-07-2938.  pmid:18483246 <ul class="reflinks" data-doi="10.1158/0008-5472.can-07-2938"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-07-2938" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Onder%5Bauthor%5D+AND+Loss+of+E-cadherin+promotes+metastasis+via+multiple+downstream+transcriptional+pathways" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Loss+of+E-cadherin+promotes+metastasis+via+multiple+downstream+transcriptional+pathways&amp;author=Onder&amp;publication_year=2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0147489.ref003" id="pone.0147489.ref003" class="link-target"></a>Cancer Cell Line Encyclopedia, 2015 (<a href="http://www.broadinstitute.org/ccle">http://www.broadinstitute.org/ccle</a>) <ul class="find-nolinks"></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0147489.ref004" id="pone.0147489.ref004" class="link-target"></a>Mani S, Guo W, Liao M, Eaton E, Ayyanan A, Zhou A et al. The epithelial-mesenchymal transition generates cells with properties of stem cells. Cell 2008; 133: 704–715.  doi: 10.1016/j.cell.2008.03.027.  pmid:18485877 <ul class="reflinks" data-doi="10.1016/j.cell.2008.03.027"><li><a href="http://dx.doi.org/10.1016/j.cell.2008.03.027" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mani%5Bauthor%5D+AND+The+epithelial-mesenchymal+transition+generates+cells+with+properties+of+stem+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=The+epithelial-mesenchymal+transition+generates+cells+with+properties+of+stem+cells&amp;author=Mani&amp;publication_year=2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0147489.ref005" id="pone.0147489.ref005" class="link-target"></a>Asiedu MK, Ingle JN, Behrens MD, Radisky DC, Knutson KL. TGFbeta/TNF(alpha)-mediated epithelial-mesenchymal transition generates breast cancer stem cells with a claudin-low phenotype. Cancer Res 2011; 71(13):4707–19.  doi: 10.1158/0008-5472.CAN-10-4554.  pmid:21555371 <ul class="reflinks" data-doi="10.1158/0008-5472.can-10-4554"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-10-4554" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asiedu%5Bauthor%5D+AND+TGFbeta%2FTNF%28alpha%29-mediated+epithelial-mesenchymal+transition+generates+breast+cancer+stem+cells+with+a+claudin-low+phenotype" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=TGFbeta%2FTNF%28alpha%29-mediated+epithelial-mesenchymal+transition+generates+breast+cancer+stem+cells+with+a+claudin-low+phenotype&amp;author=Asiedu&amp;publication_year=2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0147489.ref006" id="pone.0147489.ref006" class="link-target"></a>Nelson WJ, Nusse R. Convergence of Wnt, beta-catenin, and cadherin pathways. Science 2004; 303: 1483–1487. pmid:15001769
      doi:
      10.1126/science.1094291 <ul class="reflinks" data-doi="10.1126/science.1094291"><li><a href="http://dx.doi.org/10.1126/science.1094291" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nelson%5Bauthor%5D+AND+Convergence+of+Wnt%2C+beta-catenin%2C+and+cadherin+pathways" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Convergence+of+Wnt%2C+beta-catenin%2C+and+cadherin+pathways&amp;author=Nelson&amp;publication_year=2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0147489.ref007" id="pone.0147489.ref007" class="link-target"></a>Berx G, Cleton-Jansen AM, Strumane K, de Leeuw WJ, Nollet F, van Roy F, et al. E-cadherin is inactivated in a majority of invasive human lobular breast cancers by truncation mutations throughout its extracellular domain. Oncogene 1996; 13(9):1919–1925. pmid:8934538 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Berx&amp;atitle2=E-cadherin+is+inactivated+in+a+majority+of+invasive+human+lobular+breast+cancers+by+truncation+mutations+throughout+its+extracellular+domain&amp;auth=Berx&amp;atitle=E-cadherin+is+inactivated+in+a+majority+of+invasive+human+lobular+breast+cancers+by+truncation+mutations+throughout+its+extracellular+domain" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Berx%5Bauthor%5D+AND+E-cadherin+is+inactivated+in+a+majority+of+invasive+human+lobular+breast+cancers+by+truncation+mutations+throughout+its+extracellular+domain" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=E-cadherin+is+inactivated+in+a+majority+of+invasive+human+lobular+breast+cancers+by+truncation+mutations+throughout+its+extracellular+domain&amp;author=Berx&amp;publication_year=1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0147489.ref008" id="pone.0147489.ref008" class="link-target"></a>Lenart A, Dudkiewicz M, Grynberg M, Pawłowski K. CLCAs—a family of metalloproteases of intriguing phylogenetic distribution and with cases of substituted catalytic sites. PLoS One 2013; 8(5):e62272.  doi: 10.1371/journal.pone.0062272.  pmid:23671590 <ul class="reflinks" data-doi="10.1371/journal.pone.0062272"><li><a href="http://dx.doi.org/10.1371/journal.pone.0062272" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lenart%5Bauthor%5D+AND+CLCAs%E2%80%94a+family+of+metalloproteases+of+intriguing+phylogenetic+distribution+and+with+cases+of+substituted+catalytic+sites" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=CLCAs%E2%80%94a+family+of+metalloproteases+of+intriguing+phylogenetic+distribution+and+with+cases+of+substituted+catalytic+sites&amp;author=Lenart&amp;publication_year=2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0147489.ref009" id="pone.0147489.ref009" class="link-target"></a>Gruber A.D., Elble R.C., and Pauli B.. Discovery and cloning of the CLCA gene family. Current Topics in Membranes 2002; 53, 367–387
      doi:
      10.1016/s1063-5823(02)53042-8 <ul class="reflinks" data-doi="10.1016/s1063-5823(02)53042-8"><li><a href="http://dx.doi.org/10.1016/s1063-5823(02)53042-8" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gruber%5Bauthor%5D+AND+Discovery+and+cloning+of+the+CLCA+gene+family" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Discovery+and+cloning+of+the+CLCA+gene+family&amp;author=Gruber&amp;publication_year=2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0147489.ref010" id="pone.0147489.ref010" class="link-target"></a>Pawłowski K., Lepistö M., Meinander N., Sivars U., Varga M., Wieslander E. Novel conserved hydrolase domain in the CLCA family of alleged calcium-activated chloride channels. Proteins 2006; 63, 424–439. pmid:16470849
      doi:
      10.1002/prot.20887 <ul class="reflinks" data-doi="10.1002/prot.20887"><li><a href="http://dx.doi.org/10.1002/prot.20887" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paw%C5%82owski%5Bauthor%5D+AND+Novel+conserved+hydrolase+domain+in+the+CLCA+family+of+alleged+calcium-activated+chloride+channels" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Novel+conserved+hydrolase+domain+in+the+CLCA+family+of+alleged+calcium-activated+chloride+channels&amp;author=Paw%C5%82owski&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0147489.ref011" id="pone.0147489.ref011" class="link-target"></a>Elble RC, Walia V, Cheng HC, Connon CJ, Mundhenk L, Gruber AD, et al. The putative chloride channel hCLCA2 has a single C-terminal transmembrane segment. J Biol Chem 2006; 281(40):29448–54. pmid:16873362
      doi:
      10.1074/jbc.m605919200 <ul class="reflinks" data-doi="10.1074/jbc.m605919200"><li><a href="http://dx.doi.org/10.1074/jbc.m605919200" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elble%5Bauthor%5D+AND+The+putative+chloride+channel+hCLCA2+has+a+single+C-terminal+transmembrane+segment" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=The+putative+chloride+channel+hCLCA2+has+a+single+C-terminal+transmembrane+segment&amp;author=Elble&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0147489.ref012" id="pone.0147489.ref012" class="link-target"></a>Walia V, Ding M, Kumar S, Nie D, Premkumar LS, Elble RC. hCLCA2 is a p53-Inducible Inhibitor of Breast Cancer Cell Proliferation. Cancer Res 2009; 69, 6624–6633.  doi: 10.1158/0008-5472.CAN-08-4101.  pmid:19654313 <ul class="reflinks" data-doi="10.1158/0008-5472.can-08-4101"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-08-4101" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Walia%5Bauthor%5D+AND+hCLCA2+is+a+p53-Inducible+Inhibitor+of+Breast+Cancer+Cell+Proliferation" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=hCLCA2+is+a+p53-Inducible+Inhibitor+of+Breast+Cancer+Cell+Proliferation&amp;author=Walia&amp;publication_year=2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0147489.ref013" id="pone.0147489.ref013" class="link-target"></a>Li X, Cowell JK, Sossey-Alaoui K. CLCA2 tumour suppressor gene in 1p31 is epigenetically regulated in breast cancer. Oncogene. 2004 Feb 19;23(7):1474–80. pmid:14973555
      doi:
      10.1038/sj.onc.1207249 <ul class="reflinks" data-doi="10.1038/sj.onc.1207249"><li><a href="http://dx.doi.org/10.1038/sj.onc.1207249" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Li%5Bauthor%5D+AND+CLCA2+tumour+suppressor+gene+in+1p31+is+epigenetically+regulated+in+breast+cancer" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=CLCA2+tumour+suppressor+gene+in+1p31+is+epigenetically+regulated+in+breast+cancer&amp;author=Li&amp;publication_year=2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0147489.ref014" id="pone.0147489.ref014" class="link-target"></a>Gruber AD, Pauli BU. Tumorigenicity of human breast cancer is associated with loss of the Ca2+-activated chloride channel CLCA2. Cancer Res. 1999; 59(21):5488–5491. pmid:10554024 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Gruber&amp;atitle2=Tumorigenicity+of+human+breast+cancer+is+associated+with+loss+of+the+Ca2%2B-activated+chloride+channel+CLCA2&amp;auth=Gruber&amp;atitle=Tumorigenicity+of+human+breast+cancer+is+associated+with+loss+of+the+Ca2%2B-activated+chloride+channel+CLCA2" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gruber%5Bauthor%5D+AND+Tumorigenicity+of+human+breast+cancer+is+associated+with+loss+of+the+Ca2%2B-activated+chloride+channel+CLCA2" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Tumorigenicity+of+human+breast+cancer+is+associated+with+loss+of+the+Ca2%2B-activated+chloride+channel+CLCA2&amp;author=Gruber&amp;publication_year=1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0147489.ref015" id="pone.0147489.ref015" class="link-target"></a>Chunyan Zhao, Yichun Q, Philip J, Jian W, Li X, Pegah R, et al. Genome-wide Profiling of AP-1–Regulated Transcription Provides Insights into the Invasiveness of Triple-Negative Breast Cancer. Cancer Res 2014; 74, 3983–3994.  doi: 10.1158/0008-5472.CAN-13-3396.  pmid:24830720 <ul class="reflinks" data-doi="10.1158/0008-5472.can-13-3396"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-13-3396" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chunyan%5Bauthor%5D+AND+Genome-wide+Profiling+of+AP-1%E2%80%93Regulated+Transcription+Provides+Insights+into+the+Invasiveness+of+Triple-Negative+Breast+Cancer" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Genome-wide+Profiling+of+AP-1%E2%80%93Regulated+Transcription+Provides+Insights+into+the+Invasiveness+of+Triple-Negative+Breast+Cancer&amp;author=Chunyan&amp;publication_year=2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0147489.ref016" id="pone.0147489.ref016" class="link-target"></a>Sasaki Y, Koyama R, Maruyama R, Hirano T, Tamura M, Sugisaka J, et al. CLCA2, a target of the p53 family, negatively regulates cancer cell migration and invasion. Cancer Biol Ther 2012; 13(14):1512–1521.  doi: 10.4161/cbt.22280.  pmid:22990203 <ul class="reflinks" data-doi="10.4161/cbt.22280"><li><a href="http://dx.doi.org/10.4161/cbt.22280" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sasaki%5Bauthor%5D+AND+CLCA2%2C+a+target+of+the+p53+family%2C+negatively+regulates+cancer+cell+migration+and+invasion" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=CLCA2%2C+a+target+of+the+p53+family%2C+negatively+regulates+cancer+cell+migration+and+invasion&amp;author=Sasaki&amp;publication_year=2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0147489.ref017" id="pone.0147489.ref017" class="link-target"></a>Walia V, Yang Y, Cao D, Cheng J, Rao K, Hollier B, et al. Loss of breast epithelial marker hCLCA2 promotes epithelial to mesenchymal transition and metastasis. Oncogene 2012; 31, 2237–2246.
      doi:
      10.1038/onc.2011.392 <ul class="reflinks" data-doi="10.1038/onc.2011.392"><li><a href="http://dx.doi.org/10.1038/onc.2011.392" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Walia%5Bauthor%5D+AND+Loss+of+breast+epithelial+marker+hCLCA2+promotes+epithelial+to+mesenchymal+transition+and+metastasis" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Loss+of+breast+epithelial+marker+hCLCA2+promotes+epithelial+to+mesenchymal+transition+and+metastasis&amp;author=Walia&amp;publication_year=2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0147489.ref018" id="pone.0147489.ref018" class="link-target"></a>Sarrio D, Rodriguez-Pinilla SM, Hardisson D, Cano A, Moreno-Bueno G, Palacios J. Epithelial-mesenchymal transition in breast cancer relates to the basal-like phenotype. Cancer Res 2008; 68: 989–997.  doi: 10.1158/0008-5472.CAN-07-2017.  pmid:18281472 <ul class="reflinks" data-doi="10.1158/0008-5472.can-07-2017"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-07-2017" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sarrio%5Bauthor%5D+AND+Epithelial-mesenchymal+transition+in+breast+cancer+relates+to+the+basal-like+phenotype" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Epithelial-mesenchymal+transition+in+breast+cancer+relates+to+the+basal-like+phenotype&amp;author=Sarrio&amp;publication_year=2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0147489.ref019" id="pone.0147489.ref019" class="link-target"></a>Walia V., and Elble RC. Enrichment for breast cancer cells with stem/progenitor properties by differential adhesion. Stem Cells &amp; Development 2010; 19, 1175–1182.
      doi:
      10.1089/scd.2009.0430 <ul class="reflinks" data-doi="10.1089/scd.2009.0430"><li><a href="http://dx.doi.org/10.1089/scd.2009.0430" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Walia%5Bauthor%5D+AND+Enrichment+for+breast+cancer+cells+with+stem%2Fprogenitor+properties+by+differential+adhesion" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Enrichment+for+breast+cancer+cells+with+stem%2Fprogenitor+properties+by+differential+adhesion&amp;author=Walia&amp;publication_year=2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0147489.ref020" id="pone.0147489.ref020" class="link-target"></a>Hamann M, Gibson A, Davies N, Jowett A, Walhin JP, Partington L. Human ClCa1 modulates anionic conduction of calcium-dependent chloride currents. J Physiol 2009; 587:2255–2274.  doi: 10.1113/jphysiol.2009.170159.  pmid:19307298 <ul class="reflinks" data-doi="10.1113/jphysiol.2009.170159"><li><a href="http://dx.doi.org/10.1113/jphysiol.2009.170159" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hamann%5Bauthor%5D+AND+Human+ClCa1+modulates+anionic+conduction+of+calcium-dependent+chloride+currents" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Human+ClCa1+modulates+anionic+conduction+of+calcium-dependent+chloride+currents&amp;author=Hamann&amp;publication_year=2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0147489.ref021" id="pone.0147489.ref021" class="link-target"></a>Vrba L, Junk DJ, Novak P, Futscher BW. p53 induces distinct epigenetic states at its direct target promoters. BMC Genomics 2008; 9:486.  doi: 10.1186/1471-2164-9-486.  pmid:18922183 <ul class="reflinks" data-doi="10.1186/1471-2164-9-486"><li><a href="http://dx.doi.org/10.1186/1471-2164-9-486" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vrba%5Bauthor%5D+AND+p53+induces+distinct+epigenetic+states+at+its+direct+target+promoters" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=p53+induces+distinct+epigenetic+states+at+its+direct+target+promoters&amp;author=Vrba&amp;publication_year=2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0147489.ref022" id="pone.0147489.ref022" class="link-target"></a>Viganò MA, Lamartine J, Testoni B, Merico D, Alotto D, Castagnoli C, et al. New p63 targets in keratinocytes identified by a genome-wide approach. EMBO J 2006; 25(21):5105–16. pmid:17036050
      doi:
      10.1038/sj.emboj.7601375 <ul class="reflinks" data-doi="10.1038/sj.emboj.7601375"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601375" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vigan%C3%B2%5Bauthor%5D+AND+New+p63+targets+in+keratinocytes+identified+by+a+genome-wide+approach" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=New+p63+targets+in+keratinocytes+identified+by+a+genome-wide+approach&amp;author=Vigan%C3%B2&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0147489.ref023" id="pone.0147489.ref023" class="link-target"></a>Carroll DK, Carroll JS, Leong CO, Cheng F, Brown M, Mills AA, et al. p63 regulates an adhesion programme and cell survival in epithelial cells. Nat Cell Biol 2006; 8(6):551–61. pmid:16715076
      doi:
      10.1038/ncb1420 <ul class="reflinks" data-doi="10.1038/ncb1420"><li><a href="http://dx.doi.org/10.1038/ncb1420" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carroll%5Bauthor%5D+AND+p63+regulates+an+adhesion+programme+and+cell+survival+in+epithelial+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=p63+regulates+an+adhesion+programme+and+cell+survival+in+epithelial+cells&amp;author=Carroll&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0147489.ref024" id="pone.0147489.ref024" class="link-target"></a>Guttinger M, Sutti F, Panigada M, Porcellini S, Merati B, Mariani M, et al. Epithelial V-like antigen (EVA), a novel member of the immunoglobulin superfamily, expressed in embryonic epithelia with a potential role as homotypic adhesion molecule in thymus histogenesis. J Cell Biol 1998; 141(4):1061–71. pmid:9585423
      doi:
      10.1083/jcb.141.4.1061 <ul class="reflinks" data-doi="10.1083/jcb.141.4.1061"><li><a href="http://dx.doi.org/10.1083/jcb.141.4.1061" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guttinger%5Bauthor%5D+AND+Epithelial+V-like+antigen+%28EVA%29%2C+a+novel+member+of+the+immunoglobulin+superfamily%2C+expressed+in+embryonic+epithelia+with+a+potential+role+as+homotypic+adhesion+molecule+in+thymus+histogenesis" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Epithelial+V-like+antigen+%28EVA%29%2C+a+novel+member+of+the+immunoglobulin+superfamily%2C+expressed+in+embryonic+epithelia+with+a+potential+role+as+homotypic+adhesion+molecule+in+thymus+histogenesis&amp;author=Guttinger&amp;publication_year=1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0147489.ref025" id="pone.0147489.ref025" class="link-target"></a>Beckley JR, Pauli BU and Elble RC. Re-expression of detachment-inducible chloride channel mCLCA5 suppresses growth of metastatic breast cancer cells. J.Biol.Chem 2004; 279: 41634–41641. pmid:15292178
      doi:
      10.1074/jbc.m408334200 <ul class="reflinks" data-doi="10.1074/jbc.m408334200"><li><a href="http://dx.doi.org/10.1074/jbc.m408334200" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Beckley%5Bauthor%5D+AND+Re-expression+of+detachment-inducible+chloride+channel+mCLCA5+suppresses+growth+of+metastatic+breast+cancer+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Re-expression+of+detachment-inducible+chloride+channel+mCLCA5+suppresses+growth+of+metastatic+breast+cancer+cells&amp;author=Beckley&amp;publication_year=2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0147489.ref026" id="pone.0147489.ref026" class="link-target"></a>Yu Y, Walia V, Elble RC. Loss of CLCA4 Promotes Epithelial-to-Mesenchymal Transition in Breast Cancer Cells. PLoS ONE 2013; 8(12): e83943.  doi: 10.1371/journal.pone.0083943.  pmid:24386311 <ul class="reflinks" data-doi="10.1371/journal.pone.0083943"><li><a href="http://dx.doi.org/10.1371/journal.pone.0083943" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yu%5Bauthor%5D+AND+Loss+of+CLCA4+Promotes+Epithelial-to-Mesenchymal+Transition+in+Breast+Cancer+Cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Loss+of+CLCA4+Promotes+Epithelial-to-Mesenchymal+Transition+in+Breast+Cancer+Cells&amp;author=Yu&amp;publication_year=2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0147489.ref027" id="pone.0147489.ref027" class="link-target"></a>Wright E, Kusha R, Nicholas H, Stefan L, Carrithers M.D. Epithelial V-Like Antigen Mediates Efficacy of AntiAlpha4 Integrin Treatment in a Mouse Model of Multiple Sclerosis. PLoS ONE 2013; 8(8): e70954.  doi: 10.1371/journal.pone.0070954.  pmid:23951051 <ul class="reflinks" data-doi="10.1371/journal.pone.0070954"><li><a href="http://dx.doi.org/10.1371/journal.pone.0070954" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wright%5Bauthor%5D+AND+Epithelial+V-Like+Antigen+Mediates+Efficacy+of+AntiAlpha4+Integrin+Treatment+in+a+Mouse+Model+of+Multiple+Sclerosis" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Epithelial+V-Like+Antigen+Mediates+Efficacy+of+AntiAlpha4+Integrin+Treatment+in+a+Mouse+Model+of+Multiple+Sclerosis&amp;author=Wright&amp;publication_year=2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0147489.ref028" id="pone.0147489.ref028" class="link-target"></a>Jen-Tsan Chi, Howard YC, Guttorm H, Frode LJ, Olga GT, Dustin SC, et al. Endothelial cell diversity revealed by global expression profiling. Proc Natl Acad Sci USA 2003; 100 (19): 10623–10628. pmid:12963823 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Jen-Tsan&amp;atitle2=Endothelial+cell+diversity+revealed+by+global+expression+profiling&amp;auth=Jen-Tsan&amp;atitle=Endothelial+cell+diversity+revealed+by+global+expression+profiling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jen-Tsan%5Bauthor%5D+AND+Endothelial+cell+diversity+revealed+by+global+expression+profiling" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Endothelial+cell+diversity+revealed+by+global+expression+profiling&amp;author=Jen-Tsan&amp;publication_year=2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0147489.ref029" id="pone.0147489.ref029" class="link-target"></a>Dontu G, Wissam MA, Jessica MF, Kyle WJ, Clarke Michael F., Kawamura Mari J., et al. In vitro propagation and transcriptional profiling of human mammary stem/progenitor cells. Genes Dev. 2003;17(10):1253–1270. pmid:12756227
      doi:
      10.1101/gad.1061803 <ul class="reflinks" data-doi="10.1101/gad.1061803"><li><a href="http://dx.doi.org/10.1101/gad.1061803" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dontu%5Bauthor%5D+AND+In+vitro+propagation+and+transcriptional+profiling+of+human+mammary+stem%2Fprogenitor+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=In+vitro+propagation+and+transcriptional+profiling+of+human+mammary+stem%2Fprogenitor+cells&amp;author=Dontu&amp;publication_year=2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0147489.ref030" id="pone.0147489.ref030" class="link-target"></a>Marshall AM, Vaibhav PP, Maureen AS, Nelson DH. In vitro multipotent differentiation and barrier function of a human mammary epithelium. Cell and Tissue Res 2009; 335(2) 383–395.
      doi:
      10.1007/s00441-008-0719-0 <ul class="reflinks" data-doi="10.1007/s00441-008-0719-0"><li><a href="http://dx.doi.org/10.1007/s00441-008-0719-0" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Marshall%5Bauthor%5D+AND+In+vitro+multipotent+differentiation+and+barrier+function+of+a+human+mammary+epithelium" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=In+vitro+multipotent+differentiation+and+barrier+function+of+a+human+mammary+epithelium&amp;author=Marshall&amp;publication_year=2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0147489.ref031" id="pone.0147489.ref031" class="link-target"></a>Taube JH, Herschkowitz JI, Komurov K, Zhou AY, Gupta S, Yang J, et al. Core epithelial-to-mesenchymal transition interactome gene-expression signature is associated with claudin-low and metaplastic breast cancer subtypes. Proc Natl Acad Sci USA 2010; 107(35):15449–15454.  doi: 10.1073/pnas.1004900107.  pmid:20713713 <ul class="reflinks" data-doi="10.1073/pnas.1004900107"><li><a href="http://dx.doi.org/10.1073/pnas.1004900107" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taube%5Bauthor%5D+AND+Core+epithelial-to-mesenchymal+transition+interactome+gene-expression+signature+is+associated+with+claudin-low+and+metaplastic+breast+cancer+subtypes" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Core+epithelial-to-mesenchymal+transition+interactome+gene-expression+signature+is+associated+with+claudin-low+and+metaplastic+breast+cancer+subtypes&amp;author=Taube&amp;publication_year=2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0147489.ref032" id="pone.0147489.ref032" class="link-target"></a>Stinson S, Mark RL, Alex TA, Nancy Y, Hyo-Jin K, Carol OB, et al. TRPS1 Targeting by miR-221/222 Promotes the Epithelial-to-Mesenchymal Transition in Breast Cancer. Sci. Signal 2011; 4(177):ra41.  doi: 10.1126/scisignal.2001538.  pmid:21673316 <ul class="reflinks" data-doi="10.1126/scisignal.2001538"><li><a href="http://dx.doi.org/10.1126/scisignal.2001538" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stinson%5Bauthor%5D+AND+TRPS1+Targeting+by+miR-221%2F222+Promotes+the+Epithelial-to-Mesenchymal+Transition+in+Breast+Cancer" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=TRPS1+Targeting+by+miR-221%2F222+Promotes+the+Epithelial-to-Mesenchymal+Transition+in+Breast+Cancer&amp;author=Stinson&amp;publication_year=2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0147489.ref033" id="pone.0147489.ref033" class="link-target"></a>Miranda KC, Khromykh T, Christy P, Le TL, Gottardi CJ, Yap AS, et al. A dileucine motif targets E-cadherin to the basolateral cell surface in Madin-Darby canine kidney and LLC-PK1 epithelial cells. J Biol Chem 2001; 276(25):22565–72. pmid:11312273
      doi:
      10.1074/jbc.m101907200 <ul class="reflinks" data-doi="10.1074/jbc.m101907200"><li><a href="http://dx.doi.org/10.1074/jbc.m101907200" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Miranda%5Bauthor%5D+AND+A+dileucine+motif+targets+E-cadherin+to+the+basolateral+cell+surface+in+Madin-Darby+canine+kidney+and+LLC-PK1+epithelial+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=A+dileucine+motif+targets+E-cadherin+to+the+basolateral+cell+surface+in+Madin-Darby+canine+kidney+and+LLC-PK1+epithelial+cells&amp;author=Miranda&amp;publication_year=2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0147489.ref034" id="pone.0147489.ref034" class="link-target"></a>Ikenouchi J, Umeda K, Tsukita S, Furuse M, Tsukita S. Requirement of ZO-1 for the formation of belt-like adherens junctions during epithelial cell polarization. J Cell Biol 2007; 176(6):779–86. pmid:17353356
      doi:
      10.1083/jcb.200612080 <ul class="reflinks" data-doi="10.1083/jcb.200612080"><li><a href="http://dx.doi.org/10.1083/jcb.200612080" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ikenouchi%5Bauthor%5D+AND+Requirement+of+ZO-1+for+the+formation+of+belt-like+adherens+junctions+during+epithelial+cell+polarization" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Requirement+of+ZO-1+for+the+formation+of+belt-like+adherens+junctions+during+epithelial+cell+polarization&amp;author=Ikenouchi&amp;publication_year=2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0147489.ref035" id="pone.0147489.ref035" class="link-target"></a>Chatterjee G, Carrithers LM, Carrithers MD. Epithelial V-like antigen regulates permeability of the blood-CSF barrier. Biochem Biophys Res Commun. 2008; 372(3):412–417.  doi: 10.1016/j.bbrc.2008.05.053.  pmid:18498762 <ul class="reflinks" data-doi="10.1016/j.bbrc.2008.05.053"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2008.05.053" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chatterjee%5Bauthor%5D+AND+Epithelial+V-like+antigen+regulates+permeability+of+the+blood-CSF+barrier" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Epithelial+V-like+antigen+regulates+permeability+of+the+blood-CSF+barrier&amp;author=Chatterjee&amp;publication_year=2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0147489.ref036" id="pone.0147489.ref036" class="link-target"></a>Gonzalez-Mariscal L., Betanzos A., and Avila-Flores A.. MAGUK proteins: structure and role in the tight junction. Semin. Cell Dev. Biol 2000; 11:315–324. pmid:10966866
      doi:
      10.1006/scdb.2000.0178 <ul class="reflinks" data-doi="10.1006/scdb.2000.0178"><li><a href="http://dx.doi.org/10.1006/scdb.2000.0178" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gonzalez-Mariscal%5Bauthor%5D+AND+MAGUK+proteins%3A+structure+and+role+in+the+tight+junction" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=MAGUK+proteins%3A+structure+and+role+in+the+tight+junction&amp;author=Gonzalez-Mariscal&amp;publication_year=2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0147489.ref037" id="pone.0147489.ref037" class="link-target"></a>Katoh M, Katoh M. Cross-talk of WNT and FGF signaling pathways at GSK3beta to regulate beta-catenin and SNAIL signaling cascades. Cancer Biol Ther 2006;5(9):1059–1064. pmid:16940750
      doi:
      10.4161/cbt.5.9.3151 <ul class="reflinks" data-doi="10.4161/cbt.5.9.3151"><li><a href="http://dx.doi.org/10.4161/cbt.5.9.3151" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Katoh%5Bauthor%5D+AND+Cross-talk+of+WNT+and+FGF+signaling+pathways+at+GSK3beta+to+regulate+beta-catenin+and+SNAIL+signaling+cascades" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Cross-talk+of+WNT+and+FGF+signaling+pathways+at+GSK3beta+to+regulate+beta-catenin+and+SNAIL+signaling+cascades&amp;author=Katoh&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0147489.ref038" id="pone.0147489.ref038" class="link-target"></a>Almeida M, Han L, Bellido T, Manolagas SC, Kousteni S. Wnt proteins prevent apoptosis of both uncommitted osteoblast progenitors and differentiated osteoblasts by beta-catenin-dependent and -independent signaling cascades involving src/ERK and phosphatidylinositol 3-kinase/AKT. J Biol Chem 2005; 280:41342–41351. pmid:16251184
      doi:
      10.1074/jbc.m502168200 <ul class="reflinks" data-doi="10.1074/jbc.m502168200"><li><a href="http://dx.doi.org/10.1074/jbc.m502168200" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Almeida%5Bauthor%5D+AND+Wnt+proteins+prevent+apoptosis+of+both+uncommitted+osteoblast+progenitors+and+differentiated+osteoblasts+by+beta-catenin-dependent+and+-independent+signaling+cascades+involving+src%2FERK+and+phosphatidylinositol+3-kinase%2FAKT" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Wnt+proteins+prevent+apoptosis+of+both+uncommitted+osteoblast+progenitors+and+differentiated+osteoblasts+by+beta-catenin-dependent+and+-independent+signaling+cascades+involving+src%2FERK+and+phosphatidylinositol+3-kinase%2FAKT&amp;author=Almeida&amp;publication_year=2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0147489.ref039" id="pone.0147489.ref039" class="link-target"></a>Howel LR, Subbaramaiah K, Chung WJ, Dannenberg AJ, Brown AM. Transcriptional activation of cyclooxygenase-2 in Wnt-1- transformed mouse mammary epithelial cells. Cancer Res 1999; 59:1572–577. pmid:10197631 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Howel&amp;atitle2=Transcriptional+activation+of+cyclooxygenase-2+in+Wnt-1-+transformed+mouse+mammary+epithelial+cells&amp;auth=Howel&amp;atitle=Transcriptional+activation+of+cyclooxygenase-2+in+Wnt-1-+transformed+mouse+mammary+epithelial+cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Howel%5Bauthor%5D+AND+Transcriptional+activation+of+cyclooxygenase-2+in+Wnt-1-+transformed+mouse+mammary+epithelial+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Transcriptional+activation+of+cyclooxygenase-2+in+Wnt-1-+transformed+mouse+mammary+epithelial+cells&amp;author=Howel&amp;publication_year=1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0147489.ref040" id="pone.0147489.ref040" class="link-target"></a>Tetsu O, McCormick F. Beta-catenin regulates expression of cyclin D1 in colon carcinoma cells. Nature 1999; 398:422–426. pmid:10201372 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Tetsu&amp;atitle2=Beta-catenin+regulates+expression+of+cyclin+D1+in+colon+carcinoma+cells&amp;auth=Tetsu&amp;atitle=Beta-catenin+regulates+expression+of+cyclin+D1+in+colon+carcinoma+cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tetsu%5Bauthor%5D+AND+Beta-catenin+regulates+expression+of+cyclin+D1+in+colon+carcinoma+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Beta-catenin+regulates+expression+of+cyclin+D1+in+colon+carcinoma+cells&amp;author=Tetsu&amp;publication_year=1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0147489.ref041" id="pone.0147489.ref041" class="link-target"></a>Kolligs FT, Nieman MT, Winer I, Hu G, Van Mater D, Feng Y, et al. ITF-2, a downstream target of the Wnt/TCF pathway, is activated in human cancers with beta-catenin defects and promotes neoplastic transformation. Cancer Cell 2002; 1:145–155. pmid:12086873
      doi:
      10.1016/s1535-6108(02)00035-1 <ul class="reflinks" data-doi="10.1016/s1535-6108(02)00035-1"><li><a href="http://dx.doi.org/10.1016/s1535-6108(02)00035-1" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kolligs%5Bauthor%5D+AND+ITF-2%2C+a+downstream+target+of+the+Wnt%2FTCF+pathway%2C+is+activated+in+human+cancers+with+beta-catenin+defects+and+promotes+neoplastic+transformation" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=ITF-2%2C+a+downstream+target+of+the+Wnt%2FTCF+pathway%2C+is+activated+in+human+cancers+with+beta-catenin+defects+and+promotes+neoplastic+transformation&amp;author=Kolligs&amp;publication_year=2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0147489.ref042" id="pone.0147489.ref042" class="link-target"></a>Novellasdemunt L, Antas P, Li VS. Targeting Wnt signaling in colorectalcancer. A Review in the Theme: Cell Signaling: Proteins, Pathways and Mechanisms. Am J Physiol Cell Physiol. 2015 Oct 15;309(8):C511–21.  doi: 10.1152/ajpcell.00117.2015 Epub 2015 Aug 19. Review. pmid:26289750; PubMed Central PMCID: PMC4609654. <ul class="reflinks" data-doi="10.1152/ajpcell.00117.2015"><li><a href="http://dx.doi.org/10.1152/ajpcell.00117.2015" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novellasdemunt%5Bauthor%5D+AND+Targeting+Wnt+signaling+in+colorectalcancer.+A+Review+in+the+Theme%3A+Cell+Signaling%3A+Proteins%2C+Pathways+and+Mechanisms" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Targeting+Wnt+signaling+in+colorectalcancer.+A+Review+in+the+Theme%3A+Cell+Signaling%3A+Proteins%2C+Pathways+and+Mechanisms&amp;author=Novellasdemunt&amp;publication_year=2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0147489.ref043" id="pone.0147489.ref043" class="link-target"></a>Irie K, Shimizu K, Sakisaka T, Ikeda W, and Takai Y. Roles and modes of action of nectins in cell-cell adhesion. Semin. Cell Dev. Biol 2004; 15:643–656. pmid:15561584
      doi:
      10.1016/s1084-9521(04)00088-6 <ul class="reflinks" data-doi="10.1016/s1084-9521(04)00088-6"><li><a href="http://dx.doi.org/10.1016/s1084-9521(04)00088-6" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Irie%5Bauthor%5D+AND+Roles+and+modes+of+action+of+nectins+in+cell-cell+adhesion" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Roles+and+modes+of+action+of+nectins+in+cell-cell+adhesion&amp;author=Irie&amp;publication_year=2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0147489.ref044" id="pone.0147489.ref044" class="link-target"></a>Gumbiner B, Stevenson B., and Grimaldi A.. The role of the cell adhesion molecule vomorulin in the formation and maintenance of the epithelial junctional complex. J. Cell Biol 1988; 107:1575–1587. pmid:3049625
      doi:
      10.1083/jcb.107.4.1575 <ul class="reflinks" data-doi="10.1083/jcb.107.4.1575"><li><a href="http://dx.doi.org/10.1083/jcb.107.4.1575" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gumbiner%5Bauthor%5D+AND+The+role+of+the+cell+adhesion+molecule+vomorulin+in+the+formation+and+maintenance+of+the+epithelial+junctional+complex" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=The+role+of+the+cell+adhesion+molecule+vomorulin+in+the+formation+and+maintenance+of+the+epithelial+junctional+complex&amp;author=Gumbiner&amp;publication_year=1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0147489.ref045" id="pone.0147489.ref045" class="link-target"></a>Halbleib J.M. and Nelson W.J. Cadherins in development: cell adhesion, sorting, and tissue morphogenesis. Genes Dev 2006; 20: 3199–3214. pmid:17158740
      doi:
      10.1101/gad.1486806 <ul class="reflinks" data-doi="10.1101/gad.1486806"><li><a href="http://dx.doi.org/10.1101/gad.1486806" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Halbleib%5Bauthor%5D+AND+Cadherins+in+development%3A+cell+adhesion%2C+sorting%2C+and+tissue+morphogenesis" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Cadherins+in+development%3A+cell+adhesion%2C+sorting%2C+and+tissue+morphogenesis&amp;author=Halbleib&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0147489.ref046" id="pone.0147489.ref046" class="link-target"></a>Yonemura S, M Itoh A, Nagafuchi , and Tsukita S. Cell-to-cell adherens junction formation and actin filament organization: similarities and differences between non-polarized fibroblasts and polarized epithelial cells. J. Cell Sci 1995; 108:127–14. pmid:7738090 <ul class="reflinks"><li><a href="http://www.crossref.org/guestquery?auth2=Yonemura&amp;atitle2=Cell-to-cell+adherens+junction+formation+and+actin+filament+organization%3A+similarities+and+differences+between+non-polarized+fibroblasts+and+polarized+epithelial+cells&amp;auth=Yonemura&amp;atitle=Cell-to-cell+adherens+junction+formation+and+actin+filament+organization%3A+similarities+and+differences+between+non-polarized+fibroblasts+and+polarized+epithelial+cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yonemura%5Bauthor%5D+AND+Cell-to-cell+adherens+junction+formation+and+actin+filament+organization%3A+similarities+and+differences+between+non-polarized+fibroblasts+and+polarized+epithelial+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Cell-to-cell+adherens+junction+formation+and+actin+filament+organization%3A+similarities+and+differences+between+non-polarized+fibroblasts+and+polarized+epithelial+cells&amp;author=Yonemura&amp;publication_year=1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0147489.ref047" id="pone.0147489.ref047" class="link-target"></a>Vasioukhin V, Bauer C, Degenstein L, Wise B, Fuchs E. Hyperproliferation and defects in epithelial polarity on conditional ablation of a-catenin in skin. Cell 2001; 104:605–17. pmid:11239416
      doi:
      10.1016/s0092-8674(01)00246-x <ul class="reflinks" data-doi="10.1016/s0092-8674(01)00246-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00246-x" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vasioukhin%5Bauthor%5D+AND+Hyperproliferation+and+defects+in+epithelial+polarity+on+conditional+ablation+of+a-catenin+in+skin" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Hyperproliferation+and+defects+in+epithelial+polarity+on+conditional+ablation+of+a-catenin+in+skin&amp;author=Vasioukhin&amp;publication_year=2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0147489.ref048" id="pone.0147489.ref048" class="link-target"></a>Drees F, Pokutta S, Yamada S, Nelson WJ, and Weis WI (2005) Alpha-catenin is a molecular switch that binds E-cadherin-beta-catenin and regulates actin-filament assembly. Cell 123: 903–915. pmid:16325583
      doi:
      10.1016/j.cell.2005.09.021 <ul class="reflinks" data-doi="10.1016/j.cell.2005.09.021"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.09.021" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drees%5Bauthor%5D+AND+Alpha-catenin+is+a+molecular+switch+that+binds+E-cadherin-beta-catenin+and+regulates+actin-filament+assembly" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Alpha-catenin+is+a+molecular+switch+that+binds+E-cadherin-beta-catenin+and+regulates+actin-filament+assembly&amp;author=Drees&amp;publication_year=2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0147489.ref049" id="pone.0147489.ref049" class="link-target"></a>Brown RC, Davis TP. Calcium modulation of adherens and tight junction function: a potential mechanism for blood-brain barrier disruption after stroke. Stroke 2002; 33(6):1706–11. pmid:12053015
      doi:
      10.1161/01.str.0000016405.06729.83 <ul class="reflinks" data-doi="10.1161/01.str.0000016405.06729.83"><li><a href="http://dx.doi.org/10.1161/01.str.0000016405.06729.83" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brown%5Bauthor%5D+AND+Calcium+modulation+of+adherens+and+tight+junction+function%3A+a+potential+mechanism+for+blood-brain+barrier+disruption+after+stroke" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Calcium+modulation+of+adherens+and+tight+junction+function%3A+a+potential+mechanism+for+blood-brain+barrier+disruption+after+stroke&amp;author=Brown&amp;publication_year=2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0147489.ref050" id="pone.0147489.ref050" class="link-target"></a>Stuart RO, Sun A, Bush KT, Nigam SK. Dependence of epithelial intercellular junction biogenesis on thapsigargin-sensitive intracellular calcium stores. J Biol Chem 1996; 271(23):13636–41. pmid:8662885
      doi:
      10.1074/jbc.271.23.13636 <ul class="reflinks" data-doi="10.1074/jbc.271.23.13636"><li><a href="http://dx.doi.org/10.1074/jbc.271.23.13636" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stuart%5Bauthor%5D+AND+Dependence+of+epithelial+intercellular+junction+biogenesis+on+thapsigargin-sensitive+intracellular+calcium+stores" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Dependence+of+epithelial+intercellular+junction+biogenesis+on+thapsigargin-sensitive+intracellular+calcium+stores&amp;author=Stuart&amp;publication_year=1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0147489.ref051" id="pone.0147489.ref051" class="link-target"></a>Sala-Rabanal M, Yurtsever Z, Nichols CG, Brett TJ. Secreted CLCA1 modulates TMEM16A to activate Ca(2+)-dependent chloride currents in human cells. Elife. 2015 Mar 17;4.  doi: 10.7554/eLife.05875 pmid:25781344; PubMed Central PMCID: PMC4360653. <ul class="reflinks" data-doi="10.7554/elife.05875"><li><a href="http://dx.doi.org/10.7554/elife.05875" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sala-Rabanal%5Bauthor%5D+AND+Secreted+CLCA1+modulates+TMEM16A+to+activate+Ca%282%2B%29-dependent+chloride+currents+in+human+cells" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Secreted+CLCA1+modulates+TMEM16A+to+activate+Ca%282%2B%29-dependent+chloride+currents+in+human+cells&amp;author=Sala-Rabanal&amp;publication_year=2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0147489.ref052" id="pone.0147489.ref052" class="link-target"></a>Yurtsever Z, Sala-Rabanal M, Randolph DT, Scheaffer SM, Roswit WT, Alevy YG, Patel AC, Heier RF, Romero AG, Nichols CG, Holtzman MJ, Brett TJ. Self-cleavage of human CLCA1 protein by a novel internal metalloprotease domain controls calcium-activated chloride channel activation. J Biol Chem. 2012 Dec 7;287(50):42138–49.  doi: 10.1074/jbc.M112.410282 Epub 2012 Oct 30. pmid:23112050; PubMed Central PMCID: PMC3516759. <ul class="reflinks" data-doi="10.1074/jbc.m112.410282"><li><a href="http://dx.doi.org/10.1074/jbc.m112.410282" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yurtsever%5Bauthor%5D+AND+Self-cleavage+of+human+CLCA1+protein+by+a+novel+internal+metalloprotease+domain+controls+calcium-activated+chloride+channel+activation" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Self-cleavage+of+human+CLCA1+protein+by+a+novel+internal+metalloprotease+domain+controls+calcium-activated+chloride+channel+activation&amp;author=Yurtsever&amp;publication_year=2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0147489.ref053" id="pone.0147489.ref053" class="link-target"></a>Ching JC, Lobanova L, Loewen ME. Secreted hCLCA1 is a signaling molecule that activates airway macrophages. PLoS One. 2013 Dec 12;8(12):e83130.  doi: 10.1371/journal.pone.0083130 eCollection 2013. pmid:24349445; PubMed Central PMCID: PMC3861482. <ul class="reflinks" data-doi="10.1371/journal.pone.0083130"><li><a href="http://dx.doi.org/10.1371/journal.pone.0083130" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ching%5Bauthor%5D+AND+Secreted+hCLCA1+is+a+signaling+molecule+that+activates+airway+macrophages" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Secreted+hCLCA1+is+a+signaling+molecule+that+activates+airway+macrophages&amp;author=Ching&amp;publication_year=2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0147489.ref054" id="pone.0147489.ref054" class="link-target"></a>Shinmura K, Igarashi H, Kato H, Kawanishi Y, Inoue Y, Nakamura S, et al. CLCA2 as a novel immunohistochemical marker for differential diagnosis of squamous cell carcinoma from adenocarcinoma of the lung. Dis Markers 2014; e-pub ahead of print 7 Dec 2014;  doi: 10.1155/2014/619273.  <ul class="reflinks" data-doi="10.1155/2014/619273"><li><a href="http://dx.doi.org/10.1155/2014/619273" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shinmura%5Bauthor%5D+AND+CLCA2+as+a+novel+immunohistochemical+marker+for+differential+diagnosis+of+squamous+cell+carcinoma+from+adenocarcinoma+of+the+lung" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=CLCA2+as+a+novel+immunohistochemical+marker+for+differential+diagnosis+of+squamous+cell+carcinoma+from+adenocarcinoma+of+the+lung&amp;author=Shinmura&amp;publication_year=2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0147489.ref055" id="pone.0147489.ref055" class="link-target"></a>Connon CJ, Kawasaki S, Liles M, Koizumi N, Yamasaki K, Nakamura T, Quantock AJ, Kinoshita S. Gene expression and immunolocalisation of a calcium-activated chloride channel during the stratification of cultivated and developing corneal epithelium. Cell Tissue Res. 2006; 323(1):177–182. pmid:16158324
      doi:
      10.1007/s00441-005-0059-2 <ul class="reflinks" data-doi="10.1007/s00441-005-0059-2"><li><a href="http://dx.doi.org/10.1007/s00441-005-0059-2" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Connon%5Bauthor%5D+AND+Gene+expression+and+immunolocalisation+of+a+calcium-activated+chloride+channel+during+the+stratification+of+cultivated+and+developing+corneal+epithelium" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Gene+expression+and+immunolocalisation+of+a+calcium-activated+chloride+channel+during+the+stratification+of+cultivated+and+developing+corneal+epithelium&amp;author=Connon&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0147489.ref056" id="pone.0147489.ref056" class="link-target"></a>Canel M, Serrels A, Frame MC, Brunton VG. E-cadherin-integrin crosstalk in cancer invasion and metastasis. J Cell Sci 2013; 126: 393–401.  doi: 10.1242/jcs.100115.  pmid:23525005 <ul class="reflinks" data-doi="10.1242/jcs.100115"><li><a href="http://dx.doi.org/10.1242/jcs.100115" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Canel%5Bauthor%5D+AND+E-cadherin-integrin+crosstalk+in+cancer+invasion+and+metastasis" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=E-cadherin-integrin+crosstalk+in+cancer+invasion+and+metastasis&amp;author=Canel&amp;publication_year=2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0147489.ref057" id="pone.0147489.ref057" class="link-target"></a>Bill A, Gutierrez A, Kulkarni S, Kemp C, Bonenfant D, Voshol H, et al. ANO1 interacts with EGFR and correlates with sensitivity to EGFR-targeting therapy in head and neck cancer. Oncotarget 2015; 6(11): 9173–88. pmid:25823819
      doi:
      10.18632/oncotarget.3277 <ul class="reflinks" data-doi="10.18632/oncotarget.3277"><li><a href="http://dx.doi.org/10.18632/oncotarget.3277" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bill%5Bauthor%5D+AND+ANO1+interacts+with+EGFR+and+correlates+with+sensitivity+to+EGFR-targeting+therapy+in+head+and+neck+cancer" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=ANO1+interacts+with+EGFR+and+correlates+with+sensitivity+to+EGFR-targeting+therapy+in+head+and+neck+cancer&amp;author=Bill&amp;publication_year=2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0147489.ref058" id="pone.0147489.ref058" class="link-target"></a>Martìn-Padura I, Lostaglio S, Schneemann M, Williams L, Romano M, Fruscella P, et al. Junctional adhesion molecule, a novel member of the immunoglobulin superfamily that distributes at intercellular junctions and modulates monocyte transmigration. J Cell Biol. 1998; 142(1):117–127. pmid:9660867
      doi:
      10.1083/jcb.142.1.117 <ul class="reflinks" data-doi="10.1083/jcb.142.1.117"><li><a href="http://dx.doi.org/10.1083/jcb.142.1.117" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mart%C3%ACn-Padura%5Bauthor%5D+AND+Junctional+adhesion+molecule%2C+a+novel+member+of+the+immunoglobulin+superfamily+that+distributes+at+intercellular+junctions+and+modulates+monocyte+transmigration" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Junctional+adhesion+molecule%2C+a+novel+member+of+the+immunoglobulin+superfamily+that+distributes+at+intercellular+junctions+and+modulates+monocyte+transmigration&amp;author=Mart%C3%ACn-Padura&amp;publication_year=1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0147489.ref059" id="pone.0147489.ref059" class="link-target"></a>Charafe-Jauffret E, Ginestier C, Monville F, Finetti P, Adélaïde J, Cervera N, Fekairi S, Xerri L, Jacquemier J, Birnbaum D, Bertucci F. Gene expression profiling of breast cell lines identifies potential new basal markers. Oncogene 2006; 25(15):2273–84. pmid:16288205
      doi:
      10.1038/sj.onc.1209254 <ul class="reflinks" data-doi="10.1038/sj.onc.1209254"><li><a href="http://dx.doi.org/10.1038/sj.onc.1209254" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Charafe-Jauffret%5Bauthor%5D+AND+Gene+expression+profiling+of+breast+cell+lines+identifies+potential+new+basal+markers" target="_new" title="Go to article in PubMed">
                      PubMed/NCBI
                    </a></li><li><a href="http://scholar.google.com/scholar_lookup?title=Gene+expression+profiling+of+breast+cell+lines+identifies+potential+new+basal+markers&amp;author=Charafe-Jauffret&amp;publication_year=2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>



          <div class="ref-tooltip">
             <div class="ref_tooltip-content">

             </div>
          </div>

        </div>
      </div>
    </div>

    </section>
    <aside class="article-aside">
<!--[if IE 9]>
<style>
.dload-xml {margin-top: 38px}
</style>
<![endif]-->
<div class="dload-menu">
  <div class="dload-pdf ">
    <a href="/plosone/article/asset?id=10.1371%2Fjournal.pone.0147489.PDF"
       id="downloadPdf" target="_blank">Download PDF</a>
  </div>
  <div data-js-tooltip-hover="trigger" class="dload-hover">&nbsp;
    <ul class="dload-xml" data-js-tooltip-hover="target">
      <li><a href="/plosone/article/citation?id=10.1371%2Fjournal.pone.0147489"
             id="downloadCitation">Citation</a></li>
      <li><a href="/plosone/article/asset?id=10.1371%2Fjournal.pone.0147489.XML&download="
             id="downloadXml">XML</a>
      </li>
    </ul>

  </div>
</div>

<div class="aside-container">
<div class="print-article" id="printArticle" data-js-tooltip-hover="trigger">

  Print
  <ul class="print-options" data-js-tooltip-hover="target">
    <li>
        <a href="#" onclick="window.print(); return false;" class="preventDefault" id="printBrowser" title="Print
        Article">Print article</a>
    </li>
  


<li>
  <a title="Odyssey Press" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371%2Fjournal.pone.0147489&volume=&issue=&title=CLCA2%20Interactor%20EVA1%20Is%20Required%20for%20Mammary%20Epithelial%20Cell%20Differentiation&author_name=Grace%20Ramena%2C%20Yufang%20Yin%2C%20Yang%20Yu%2C%20Vijay%20Walia%2C%20Randolph%20C.%20Elble&start_page=1
&end_page=21">    EzReprint
  </a>
</li>

  </ul>
</div>
<div class="share-article" id="shareArticle" data-js-tooltip-hover="trigger">
  Share
  <ul data-js-tooltip-hover="target" class="share-options">

    <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489" id="shareReddit" target="_blank" title="Submit to Reddit"><img src="/plosone/resource/img/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

    <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489" id="shareGoogle" target="_blank" title="Share on Google+"><img src="/plosone/resource/img/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

    <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489"  id="shareStumble" target="_blank" title="Add to StumbleUpon"><img src="/plosone/resource/img/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

    <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489&t=CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" id="shareFacebook" target="_blank" title="Share on Facebook"><img src="/plosone/resource/img/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>


<li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489&title=CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation&summary=Checkout this article I found at PLOS"  id="shareLinkedIn" target="_blank" title="Add to LinkedIn"><img src="/plosone/resource/img/icon.linkedin.16.png" width="16" height="16" alt="LinkedIn">LinkedIn</a></li>

<li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489&title=CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation"  id="shareCiteULike" target="_blank" title="Add to CiteULike"><img src="/plosone/resource/img/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

<li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489"  id="shareMendeley" target="_blank" title="Add to Mendeley"><img src="/plosone/resource/img/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

<li><a href="https://www.pubchase.com/library?add_aid=10.1371/journal.pone.0147489&source=plos"  id="sharePubChase" target="_blank" title="Add to PubChase"><img src="/plosone/resource/img/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>

  <li><a href="http://twitter.com/intent/tweet?url=http%3A%2F%2Fdx.doi.org%2F10.1371%2Fjournal.pone.0147489&text=%23PLOSONE%3A%20CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation" target="_blank" title="share on Twitter" id="twitter-share-link"><img src="/plosone/resource/img/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

<li><a href="/plosone/article/email?id=10.1371%2Fjournal.pone.0147489" id="shareEmail" title="Email this article"><img src="/plosone/resource/img/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>

  </ul>
</div></div>

<!-- Start CrossMark Snippet v1.4 -->
<div class="crossmark-container">
  <a id="open-crossmark" href="#"><img id="crossmark-icon" src="/plosone/resource/img/logo-crossmark-bw.png" /></a>
</div>


<!-- End CrossMark Snippet -->





<div class="subject-areas-container">
  <h3>Subject Areas <div id="subjInfo">?</div>
    <div id="subjInfoText">
        <p>For more information about PLOS Subject Areas, click
          <a href="/plosone/s/help-using-this-site#loc-subject-areas">here</a>.</p>
      <span class="inline-intro">We want your feedback.</span> Do these Subject Areas make sense for this article? Click the target next to the incorrect Subject Area and let us know. Thanks for your help!


    </div>
  </h3>
  <ul id="subjectList">
      <li>
              <a class="taxo-term" title="Search for articles about Catenins"
                 href="/plosone/search?filterSubjects=Catenins&filterJournals=PLoSONE&q=">Catenins</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Catenins"><p class="taxo-explain">Is the Subject Area <strong>"Catenins"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Breast cancer"
                 href="/plosone/search?filterSubjects=Breast+cancer&filterJournals=PLoSONE&q=">Breast cancer</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Breast cancer"><p class="taxo-explain">Is the Subject Area <strong>"Breast cancer"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Immunoprecipitation"
                 href="/plosone/search?filterSubjects=Immunoprecipitation&filterJournals=PLoSONE&q=">Immunoprecipitation</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Immunoprecipitation"><p class="taxo-explain">Is the Subject Area <strong>"Immunoprecipitation"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Cell differentiation"
                 href="/plosone/search?filterSubjects=Cell+differentiation&filterJournals=PLoSONE&q=">Cell differentiation</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Cell differentiation"><p class="taxo-explain">Is the Subject Area <strong>"Cell differentiation"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Protein interactions"
                 href="/plosone/search?filterSubjects=Protein+interactions&filterJournals=PLoSONE&q=">Protein interactions</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Protein interactions"><p class="taxo-explain">Is the Subject Area <strong>"Protein interactions"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Breast tumors"
                 href="/plosone/search?filterSubjects=Breast+tumors&filterJournals=PLoSONE&q=">Breast tumors</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Breast tumors"><p class="taxo-explain">Is the Subject Area <strong>"Breast tumors"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Invasive ductal carcinoma"
                 href="/plosone/search?filterSubjects=Invasive+ductal+carcinoma&filterJournals=PLoSONE&q=">Invasive ductal carcinoma</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Invasive ductal carcinoma"><p class="taxo-explain">Is the Subject Area <strong>"Invasive ductal carcinoma"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Epithelial cells"
                 href="/plosone/search?filterSubjects=Epithelial+cells&filterJournals=PLoSONE&q=">Epithelial cells</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Epithelial cells"><p class="taxo-explain">Is the Subject Area <strong>"Epithelial cells"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
  </ul>
</div>
<div id="subjectErrors"></div>

<div class="skyscraper-container">
  <div class="title">Advertisement</div>
  <!-- DoubleClick Ad Zone -->
  <div class='advertisement' id='div-gpt-ad-1458247671871-1' style='width:160px; height:600px;'>
    <script type='text/javascript'>
      googletag.cmd.push(function() { googletag.display('div-gpt-ad-1458247671871-1'); });
    </script>
  </div>
</div>

<div class="twitter-container">
  <h3>Archived Tweets</h3>
  <ul id="tweetList">

  </ul>
  <div class="load-more">Load more <span></span></div>
  <div class="view-all"><a href="http://alm.plos.org/works/doi.org/10.1371/journal.pone.0147489?source_id=twitter">View all tweets</a>
  </div>
</div>

<script type="text/template" id="twitterModuleItemTemplate">
  <% _.each(items, function(item) { %>
    <li>
      <div class="tweet-info">
        <a href="http://twitter.com/<%= item.user %>">
          <span class="imgholder">
            <img class="imgLoad" src="<%= item.user_profile_image %>">
          </span>
          <div class="tweetDate"><%= item.created_at %></div>
          <div class="tweetUser">
            <strong><%= item.user_name %></strong>
            <span>@<%= item.user %></span>
          </div>
        </a>
      </div>
      <div class="tweetText">
        <%= item.text %>
      </div>
      <div id="tweetActions">
        <a class="tweet-reply" href="https://twitter.com/intent/tweet?in_reply_to<%= item.id %>&amp;text=@<%= item.user %>">
          <div>&nbsp;</div> Reply
        </a>
        <a class="tweet-retweet" href="https://twitter.com/intent/retweet?tweet_id=<%= item.id %>">
          <div>&nbsp;</div> Retweet
        </a>
        <a class="tweet-favorite" href="https://twitter.com/intent/favorite?tweet_id=<%= item.id %>">
          <div>&nbsp;</div> Favorite
        </a>
      </div>
    </li>
  <% }); %>
</script>

    </aside>
  </div>



</main>

<footer id="pageftr">
  <div class="row">
    <div class="brand-column">
      <img src="/plosone/resource/img/logo-plos-footer.png" alt="PLOS" class="logo-footer"/>
      <p class="nav-special">
&nbsp;
<!--
  Webapp build:  2.2.2 at 2016-08-04 10:54 by root, commit: 
  Service build: 1.5.2 at 2016-07-20 10:17 by root, commit: 
  Enabled dev features: []
  -->
      </p>

<ul class="nav-aux">
  <li><a href="https://www.plos.org/privacy-policy" id="ftr-privacy">Privacy Policy</a></li>
  <li><a href="https://www.plos.org/terms-of-use" id="ftr-terms">Terms of Use</a></li>
  <li><a href="https://www.plos.org/advertise/" id="ftr-advertise">Advertise</a></li>
  <li><a href="https://www.plos.org/media-inquiries" id="ftr-media">Media Inquiries</a></li>
</ul>
    </div>
    <div class="link-column">
<p class="nav-special"><a href="https://www.plos.org/publications/journals/">Publications</a></p>

<div class="nav">
  <ul>
    <li><a href="/plosbiology/" id="ftr-bio">PLOS Biology</a></li>
    <li><a href="/plosmedicine/" id="ftr-med">PLOS Medicine</a></li>
    <li><a href="/ploscompbiol/" id="ftr-compbio">PLOS Computational Biology</a></li>
    <li><a href="http://currents.plos.org" id="ftr-cur">PLOS Currents</a></li>
    <li><a href="/plosgenetics/" id="ftr-gen">PLOS Genetics</a></li>
    <li><a href="/plospathogens/" id="ftr-path">PLOS Pathogens</a></li>
    <li><a href="/plosone/" id="ftr-one">PLOS ONE</a></li>
    <li><a href="/plosntds/" id="ftr-ntds">PLOS Neglected Tropical Diseases</a></li>
  </ul>
</div>
    </div>
    <div class="link-column">
<div class="nav nav-special">
  <p><a href="https://www.plos.org" id="ftr-home">plos.org</a></p>

  <p><a href="http://blogs.plos.org" id="ftr-blog">Blogs</a></p>

  <p><a href="http://collections.plos.org" id="ftr-collections">Collections</a></p>


  <p><a href="/plosone/feedback" id="ftr-feedback">Send us feedback</a></p>
  <p><a href="/plosone/s/help-using-this-site"
        id="ftr-help">Help using this site</a></p>
  <p><a href="/plosone/lockss-manifest" id="ftr-lockss">LOCKSS</a></p>
  <p>California (US) corporation #C2354500, based in San Francisco</p>
</div>
    </div>
  </div>


</footer>


<script type="text/javascript">
  var ArticleData = {
    doi: '10.1371/journal.pone.0147489',
    title: 'CLCA2 Interactor EVA1 Is Required for Mammary Epithelial Cell Differentiation',
    date: 'Mar 01, 2016'
  };
</script>




<script type="text/javascript" async src="//platform.twitter.com/widgets.js"></script>
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js" ></script>
<script type="text/javascript" src="http://crossmark.crossref.org/javascripts/v1.5/crossmark.min.js"></script>



<!-- This file should be loaded before the renderJs, to avoid conflicts with the FigShare, that implements the MathJax also. -->

<!--  mathjax configuration options  -->
<!-- more can be found at http://docs.mathjax.org/en/latest/ -->
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  "HTML-CSS": {
    scale: 100,
    availableFonts: ["STIX","TeX"],
    preferredFont: "STIX",
    webFont: "STIX-Web",
    linebreaks: { automatic: false }
  },
  jax: ["input/MathML", "output/HTML-CSS"]
});
</script>

<script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML"></script>

<script src="/plosone/resource/compiled/asset_SQMVL4KTTK5FAFYW76CROPGT6DVXE3U4.js"></script>

  <script type="text/javascript">

    (function ($) {

      /*filesizetable*/
      $('#artText').populateFileSizes({
"info:doi/10.1371/journal.pone.0147489.g001": {
"original": 306262,
  "small": 2898,
  "inline": 18127,
  "medium": 41650,
  "large": 185295
},
"info:doi/10.1371/journal.pone.0147489.g002": {
"original": 496484,
  "small": 4391,
  "inline": 25746,
  "medium": 53295,
  "large": 200376
},
"info:doi/10.1371/journal.pone.0147489.g003": {
"original": 615490,
  "small": 4530,
  "inline": 31896,
  "medium": 74822,
  "large": 342139
},
"info:doi/10.1371/journal.pone.0147489.g004": {
"original": 2878390,
  "small": 9344,
  "inline": 86786,
  "medium": 225184,
  "large": 1604956
},
"info:doi/10.1371/journal.pone.0147489.g005": {
"original": 1090560,
  "small": 5070,
  "inline": 43525,
  "medium": 113249,
  "large": 673881
},
"info:doi/10.1371/journal.pone.0147489.t001": {
"original": 203940,
  "small": 3301,
  "inline": 23811,
  "medium": 53149,
  "large": 48723
},
"info:doi/10.1371/journal.pone.0147489.g006": {
"original": 727518,
  "small": 3414,
  "inline": 22543,
  "medium": 63796,
  "large": 458303
},
"info:doi/10.1371/journal.pone.0147489.g007": {
"original": 1301474,
  "small": 5954,
  "inline": 57472,
  "medium": 159502,
  "large": 1152751
},
"info:doi/10.1371/journal.pone.0147489.g008": {
"original": 5225606,
  "small": 10178,
  "inline": 154396,
  "medium": 487658,
  "large": 4887871
},
"info:doi/10.1371/journal.pone.0147489.g009": {
"original": 5212288,
  "small": 12141,
  "inline": 196279,
  "medium": 556101,
  "large": 5655375
},
"info:doi/10.1371/journal.pone.0147489.g010": {
"original": 1972646,
  "small": 4484,
  "inline": 68123,
  "medium": 230227,
  "large": 2274051
},
"info:doi/10.1371/journal.pone.0147489.g011": {
"original": 608338,
  "small": 4287,
  "inline": 33911,
  "medium": 80070,
  "large": 376908
},
"info:doi/10.1371/journal.pone.0147489.g012": {
"original": 512396,
  "small": 3068,
  "inline": 19606,
  "medium": 47149,
  "large": 199571
},
"info:doi/10.1371/journal.pone.0147489.g013": {
"original": 908842,
  "small": 7289,
  "inline": 54524,
  "medium": 73774,
  "large": 364810
}
}
);

    })(jQuery);

  </script>


<div id="crossmark-dialog" style="display: none;" title="">
  <!-- the external CrossMark data is loaded inside this iframe -->
  <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div><div class="reveal-modal-bg"></div>
</body>
</html>
